\documentclass[11pt]{article}
% Preamble
\usepackage[sc]{mathpazo} %Like Palatino with extensive math support
\usepackage{fullpage}
\usepackage[authoryear,sectionbib,sort]{natbib}
\bibliographystyle{amnatnat.bst}
\setlength{\bibsep}{0.0pt}
\linespread{1.7}
\usepackage[utf8]{inputenc}
\usepackage[left]{lineno}
\usepackage{titlesec}
\usepackage{amsmath}
\usepackage{amsfonts}
\usepackage{amssymb}
\usepackage[utf8]{inputenc}
\usepackage{color,soul}
\usepackage{booktabs}
\usepackage{tikz}
\usepackage{pdflscape}
\usepackage{nameref}
%\usepackage[title,titletoc,toc]{appendix}
\usepackage[colorlinks=true, allcolors=black]{hyperref}

% Section Header Formats
\titleformat{\section}[block]{\Large\bfseries\filcenter}{\thesection}{1em}{}
\titleformat{\subsection}[block]{\Large\itshape\filcenter}{\thesubsection}{1em}{}
\titleformat{\subsubsection}[block]{\large\itshape}{\thesubsubsection}{1em}{}
\titleformat{\paragraph}[runin]{\itshape}{\theparagraph}{1em}{}[. ]\renewcommand{\refname}{Literature Cited}

% Special Math Characters
\newcommand\encircle[1]{%
  \tikz[baseline=(X.base)] 
    \node (X) [draw, shape=circle, inner sep=0] {\strut #1};}


% Graphics package
\usepackage{graphicx}
\graphicspath{{../output/figs/}.pdf}

% Change default margins
\usepackage[top=0.75in, bottom=0.75in, left=0.75in, right=0.75in]{geometry}

% Definitions
\def\mathbi#1{\textbf{\em #1}}
\def\mbf#1{\mathbf{#1}}
\def\mbb#1{\mathbb{#1}}
\def\mcal#1{\mathcal{#1}}
\newcommand{\bo}[1]{{\bf #1}}
\newcommand{\tr}{{\mbox{\tiny \sf T}}}
\newcommand{\bm}[1]{\mbox{\boldmath $#1$}}
\newcommand{\om}{\omega}
\DeclareRobustCommand{\firstsecond}[2]{#2}
 \def\linenumberfont{\normalfont\scriptsize}
 \newcommand{\kron}{\otimes}


 %===========================================
% Basic info from Journal

%%%%%%%%%%%%%%%%
% Line numbering
%%%%%%%%%%%%%%%%

% Please use line numbering with your initial submission and
% subsequent revisions. After acceptance, please turn line numbering
% off by adding percent signs to the lines %\usepackage{lineno} and
% to %\linenumbers{} and %\modulolinenumbers[3] below.
%
% To avoid line numbering being thrown off around math environments,
% the math environments have to be wrapped using
% \begin{linenomath*} and \end{linenomath*}
%
% (Thanks to Vlastimil Krivan for pointing this out to us!)

%%%%%%%%%%%%
% Authorship
%%%%%%%%%%%%
% Please remove authorship information while your paper is under review,
% unless you wish to waive your anonymity under double-blind review. You
% will need to add this information back in to your final files after
% acceptance.

% The journal does not have numbered sections in the main portion of
% articles. Please refrain from using section references (Ã  la
% section~\ref{section:CountingOwlEggs}), and refer to sections by name
% (e.g. section ``Counting Owl Eggs'').

% You may wish to remove the Acknowledgments section while your paper 
% is under review (unless you wish to waive your anonymity under
% double-blind review) if the Acknowledgments reveal your identity.
% If you remove this section, you will need to add it back in to your
% final files after acceptance.

%If you have deposited data to Dryad, you should cite them somewhere in the main text (usually in the Methods or Results sections). A sentence like the following will do. All data are available in the Dryad Digital Repository (\citealt{CookEtAl2015}).

%===========================================

% This version of the LaTeX template was last updated on
% November 8, 2019.

\begin{document}
\title{Evolutionary demography and the maintenance of sexually antagonistic polymorphism in outcrossers and simultaneous hermaphrodites}
\author{Colin Olito$^{1,\ast}$ \\ 
Charlotte de Vries$^{2}$}
\date{\today}
\maketitle

\noindent{} 1. Department of Biology, Lund University, Lund 223 62, Sweden;

\noindent{} 2.  Department of Evolutionary Biology and Environmental Studies, University of Zurich, Zurich CH-8057, Switzerland;

\noindent{} $\ast$ Corresponding author; e-mail: colin.olito@gmail.com

\bigskip

\textit{Manuscript elements}: Figure~1, figure~2, table~1, online appendices~A and B (including figure~A1 and figure~A2). Figure~2 is to print in color.

\bigskip

\textit{Keywords}: Intralocus sexual conflict, Evolutionary Demography, Balancing selection, Hermaphrodite, mixed mating systems, inbreeding depression 

\bigskip

\textit{Manuscript type}: Article. %Or e-article, note, e-note, natural history miscellany, e-natural history miscellany, comment, reply, invited symposium, or historical perspective.

\bigskip

\noindent{\footnotesize Prepared using the suggested \LaTeX{} template for \textit{Am.\ Nat.}}

\linenumbers{}
\modulolinenumbers[3]

\newpage{}


%====================
% Begin Main Text
%====================

%\section{Outline}

%OK, just putting some ideas out here based roughly on our conversations about an {\itshape AmNat}-style article, the results we have so far, and what I think we should be able to do with data from COMPADRE. 

%\begin{enumerate}
	%\item {\bf Introduction}
	%\begin{enumerate}
%		\item Briefly introduce sexual antagonism and intra-locus sexual conflict. (Colin)
%		\item Simultaneous hermaphrodites, mixed mating systems, unique predictions for SA polymorphism. (Colin)
%		\item Introduce Evolutionary Demography, pivot to focus on the consequences of taking demography into account explicitly for the maintenance of SA polymorphism. (Lotte)
%		\item Introduce key difference between existing population genetic models of SA polymorphism and our models: possibility of diverse population and evolutionary dynamics. (Lotte then Colin)
%		\item Explain and highlight "demographically viable" parameter space, indicate that we are interested in whether parameter space predicted to be polymorphic by Pop.~gen.~models is, indeed, "demographically viable" (Lotte then Colin)
%		\begin{enumerate}
%			\item Pop. Gen. models tacitly assume stable population size, parameterize genotypic fitnesses relative to the 'best performing' genotype. Non-overlapping generations. Casually define fitness as lifetime reproductive fitness of a given genotype.
%			\item "In practice, these genotypic fitnesses emerge from myriad process acting throughout the life-history of individuals"
%			\item these processes contribute to key demographic rates such as the intrinsic population growth rate, 
%			\item 
%		\end{enumerate}
%		\item Building upon the two-sex stage-structured matrix models developed by \citet{deVriesCaswell2019a,deVriesCaswell2019b}, we develop stage-structured matrix models that can accommodate selection through male and female function in simultaneous hermaphrodites with mixed mating systems. The models also allow for the effects of inbreeding depression to manifest in several demographic parameters, allowing us to explore the fitness consequences of early- vs.~late-acting inbreeding depression. (Colin then Lotte)
	%\end{enumerate}
	%\item {\bf Methods}
%		\begin{enumerate}
%			\item Citation heavy review of de Vries \& Caswell AmNat (2018) \& de Vries \& Caswell AmNat (2019) TheorPopBiol
%			\item Briefly describe extension to simultaneous hermaphrodites, citing Jordan \& Connallon (2014), Abbot \& Tazzyman (2014), Olito (2017). 
%			\item Breakdown into individuals produced by selfing vs.~outcrossing.
%			\item Brief anatomy of $\mathbf{\tilde{A}}$.
%			\item Analysis. Invasion analysis \& abridged derivation of invasion conditions \& identification of extinction threholds.
%			\item Cite Appendix with full derivation distilled from notes file.
%		\end{enumerate}
	%\item {\bf When is polymorphism demographically viable?}
	%\begin{enumerate}
%		\item Review previous predictions for outcrossers \& partial selfing
%		\begin{enumerate}
%			\item Greater scope for SA polymorphism in outcrossers vs.~selfers
%			\item Greater scope for SA polymorphism when SA fitness effects are recessive (dominance reversal).
%			\item but our model allows us to see where/when invasion of SA alleles can drive the population extinct
%			\item \textbf{Fig.1 - illustration of funnel plots w/ extinction isoclines}
%		\end{enumerate}
%		\item Key results summarized in \textbf{Fig.~2}: 
%		\begin{enumerate}
%			\item For simplicity, temporarily assume no inbreeding depression ($\delta = 0$).
%			\item When populations do not have high growth-rates (i.e., when $\lambda \approx 1$), much of of polymorphic parameter space becomes demographically inviable.
%			\item Specifically, would-be polymorphic paramter space becomes inviable when segregating male-beneficial/female-deleterious alleles can reduce fecundity enough to drive the population extinct.
%			\item This effect is most dramatic for separate-sexed species and predominant outcrossers. Selfing shelters the population from the female-deleterious fitness effects because there is no opportunity for selection via male function. This can also be thought of as reproductive assurance in our model....
%			\item Convergence to pop.~gen.~predictions when $f$ is big.
%		\end{enumerate}
	%\end{enumerate}
	%\item {\bf Effects of inbreeding depression}
	%\begin{enumerate}
%		\item Review effect of I.D.~in population genetic models: causes selfing populations to look more like outcrossing ones (at least in terms of where SA polymorphism is maintained). This is because a fraction of selfed ovules are aborted, meaning the overall ratio of outcrossed vs.~selfed progeny increases.
%		\item Matrix model allows us to explore effects of I.D. in different parts of the life-cycle (early- vs.~late-acting I.D.). Point out that previous pop.~gen.~models only address early-acting I.D. via ovule viability.
%		\item MAKE CLEAR that purging of deleterious recessives thought to cause I.D. is already included in the model results presented in Fig.~3
%		\item Key results summarized in Fig.~3
%		\begin{enumerate}
%			\item Pop.~Gen.~models predict that higher early-acting ID should lead to greater proportion of polymorphic parameter space, while our models predict that we get a critical threshold of selfing at which population viability plummets.
%			\item When and how I.D. influences fitness during the life-cyclecan change the location and steepness of the decline in polymorphic parameter space.
%			\item Critically, largest deviations from pop.~gen.~predictions happen for predominantly selfing populations.
%			\item A perhaps obvious, but sometimes overlooked implication of our model predictions: selfing can offer reproductive assurance, and SA polymorphisms can be maintained in a non-trivial proportion of parameter space in partially selfing populations... BUT demographically, the population must be able to "afford" the loss of selfed ovules/offspring.
%			\item HIGHLIGHT that this result holds even though we incorporated decrease of the mutation load due to inbreeding (and corresponding decrease in I.D.)...	
%		\end{enumerate}
	%\end{enumerate}
	%\item {\bf Mimulus as a potential real world example}
	%\begin{enumerate}
%		\item We can parameterize our model with real populations' demographic rates, and make explicit predictions regarding the opportunity for SA polymorphism given the species' demography. We provide two illustrative examples:
%		\item Opportunities for SA polymorphism in {\itshape Mimulus guttatus} (now {\itshape Erythranthe guttata})
%		\item Possibly use data from \citet{PetersonEtAl2016} Eagle Meadows population, which is the 'local' population in their big common garden local adaptation expeirment.
%		\begin{enumerate}
%			\item Illustrate funnel plot using {\itshape E.~gutatta} demographic rates.
%			\item \textbf{Fig.~4}: PLOT inversion from \citet{LeeKelly2015}, on that funnel!
%		\end{enumerate}
	%\end{enumerate}
	%\item Discussion
%		\begin{enumerate}
%			\item Classic prediction from demography that most species should have a $\lambda \approx 1$
%			\item If true, the demographic impacts on the opportunities for the maintenance of sexually antagonistic polymorphisms may be pervasive...
%			\item Condition dependence, fecundity, scope for sexual confict/antagonistic polymorphism?
%			\item Do sexually dimorphic species exhibit demographic rates that increase opportunities for SA polymorphism?
%		\end{enumerate}
%\end{enumerate}




\tableofcontents
%%%%%%%%%%%%%%%%%%%%
\newpage{}
\section{Abstract}
%%%%%%%%%%%%%%%%%%%%
\ldots 

\newpage{}

%%%%%%%%%%%%%%%%%%%%%%%%
\section{Introduction}
%%%%%%%%%%%%%%%%%%%%%%%%


Population persistence or extinction depends heavily on the ability of a species to adapt to the environment. Yet, the process of adaptation can be impeded by a variety of genetic and environmental factors, such as deleterious mutations \citep{Haldane1957}, environmental shifts \citep{Maynard-Smith1976, LandeShannon1996,OrrUnckless2008}, maladaptive gene-flow \citep{KirkpatrickBarton1997, BolnickNosil2007}, or genetic constraints \citep{ConnallonHall2018}. Genetic constraints on adaptation can arise as a consequence of conflicting selection and gene-flow between different classes of individuals, or more generally, when there are genetic trade-offs between fitness components \citep{CharlesworthHughes2000, ConnallonHall2018}. 

 % Whether a population persists or goes extinct depends in large part on its ability to adapt to the environment. Yet, the process of adaptation can be impeded by a variety of genetic and environmental factors, including deleterious mutations \citep{Haldane1957}, environmental shifts causing a population to chase a new phenotypic optimum \citep{Maynard-Smith1976, LandeShannon1996,OrrUnckless2008}, maladaptive gene-flow \citep{KirkpatrickBarton1997, BolnickNosil2007}, as well as genetic constraints arising from intrinsic features of the inheritance system \citep{ConnallonHall2018}. 


%Conflicting selection and gene-flow between different classes of individuals, or genetic trade-offs between fitness components, can impose constraints on adaptation with particularly interesting evolutionary consequences \citep{CharlesworthHughes2000, ConnallonHall2018}. 
Fitness trade-offs constraining adaptation have particularly interesting evolutionary consequences. On the one hand, they prevent individuals (or classes of individuals) from reaching their phenotypic optimum in one or more fitness components, which can increase the population's overall extinction risk \citep{kokko2003sexy,harts2014demography}. On the other hand, they provide an effective mechanism for the maintenance of genetic variation, which influences the population's capactiy for future adaptation \citep{Fisher1930, CharlesworthHughes2000, ConnallonHall2018, MatthewsConnallon2019}. Hence, for traits under conflicting selection, the nature and extent of genetic variation observed in natural populations should reflect a balance between the contribution of genetic constraints to the maintainance of genetic polymorphisms, and the population dynamical consequences of the resulting maladaptation.

Sexually antagonistic selection, due to genetic trade-offs between male and female fitness, is a common feature of sexually reproducing populations, and is thought to contribute not only to the maintenance of genetic variation, but also to the evolution of sexual dimorphism and gender-related traits \citep{Lande1980, Rice1992, Charlesworth1999, RiceChippindale2001, BondurianskyChenoweth2009,Olito2019}. When such genetic trade-offs occur, "sexually antagonistic" selection (abbreviated SA hereafter) can arise whereby beneficial alleles for one sex are deleterious when expressed in the other \citep{Kidwell1977, Rice1992, ConnallonClark2012}. Alleles with opposing fitness effects through male and female sex functions can cause analogous genetic constraints on fitness in hermaphrodite populations, where both maternal and paternal reproductive success contribute jointly to each individuals' overall fitness \citep{LloydWebb1986, WebbLloyd1986, Abbott2011, JordanConnallon2014}. 


The evolutionary dynamics of sexually antagonistic alleles in hermaphrodites differ from those in dioecious populations because individuals may reproduce through a combination of self- and outcross fertilization \citep{Goodwillie2005, Igic2006, JarneAuld2006}. For example, self-fertilization is predicted to reduce the total parameter space where balanced SA polymorphisms can be maintained, while simultaneously creating a bias in selection through the female sex function (\citealt{JordanConnallon2014}; but see \citealt{Tazzyman2015}). Yet, other processes, such as genetic linkage to other SA loci or a sex-determining region \citep{Otto2011, JordanCharlesworth2012, Olito2017, Olito2019}, or spatial heterogeneity and the complexity of the life-cycle can expand the parameter space for SA polymorphism \citep{Olito-etal-2018,ConnallonSharmaOlito2019}. Overall, both theoretical predictions and current empirical data suggest there is ample scope for SA trade-offs and the maintenance of SA polymorphisms in both dioecious and hermaphrodite populations \citep{Abbott2011, WangBarrett2020}, although identifying specific SA loci from genome sequence data remains challenging \citep{RuzickaESEB2020}.


Most population genetic models, including models of the maintenance of SA polymorphisms, keep track of the relative fitnesses of genotypes, and therefore do not consider the population dynamical consequences of evolutionary change. To take the population dynamical consequences into account, a model is required that links changes in genotype distributions to population dynamics.  \cite{harts2014demography} build such a model using an individual based approach. Their model tracks both the population dynamics and the spread of sexually antagonistic genes as well as dispersal genes in a species with non-overlapping generations. \cite{deVriesCaswell2019b} introduced a model with overlapping generations, intralocus sexual antagonism, and population dynamics but did not perform analysis of the consequences of demographic viability for the scope of sexual antagonism to maintain polymorphism. 

In this paper, we extend the model by \cite{deVriesCaswell2019b} to include both dioecious and hermaphroditic species, and we use this model to study when the parameter conditions predicted to be polymorphic by population genetics models are also demographically viable. A major strength of the model is that we can use empirically estimated demographic rates to parameterize the model, enabling us to make predictions about the scope for demographically viable SA polymorphisms that are grounded in the biology of real populations. We demonstrate this with a case study of {\itshape Mimulus guttatus} (now {\itshape Erythranthe guttata})



%Such an eco-evolutionary model requires a map from genotype to phenotype, from phenotype to individual demographic rates (birth and death rates), and finally from demographic rates to population level characteristics \citep{coulson2006putting}.    
%Genetic trade-offs between fitness components due to pleiotropy represent a form of genetic constraint with particularly interesting evolutionary consequences because they act as a double-edged sword




% \bigskip

% \begin{enumerate}
% 	\item Introduce Evolutionary Demography, pivot to focus on the consequences of taking demography into account explicitly for the maintenance of SA polymorphism. [Lotte]
% 	\item Introduce key difference between existing population genetic models of SA polymorphism and our models: possibility of diverse population and evolutionary dynamics. [Lotte then Colin]
% 	\item Explain and highlight "demographically viable" parameter space, indicate that we are interested in whether parameter space predicted to be polymorphic by Pop.~gen.~models is, indeed, "demographically viable" [Lotte then Colin]
% 		\begin{enumerate}
% 			\item Pop. Gen. models tacitly assume large and/or stable population size, parameterize genotypic fitnesses relative to the 'best performing' genotype. Non-overlapping generations. Casually define fitness as lifetime reproductive fitness of a given genotype.
% 			\item "In practice, these genotypic fitnesses emerge from myriad process acting throughout the life-history of individuals"
% 			\item these processes contribute to key demographic rates such as the intrinsic population growth rate, 
% 		\end{enumerate}
% 		\item A major strength of the model is that we can use empirically estimated demographic rates to parameterize the model, enabling us to make predictions about the scope for demographically viable SA polymorphisms that are grounded in the biology of real populations.
% \end{enumerate}
% \bigskip



%%%%%%%%%%%%%%%%%%%%%%%%
\section{The Model}
%%%%%%%%%%%%%%%%%%%%%%%%

Here, we briefly describe a matrix model incorporating multiple life-cycle stages and a single diallelic locus under SA selection on male and female fertility for populations of simultaneous hermaphrodites with outcrossing or mixed-mating systems. The derivation and some of the key results follow closely those presented in \citet{deVriesCaswell2019a} and \citet{deVriesCaswell2019b}; and in fact, the model presented here reduces to the two-sex stage-structured model of \citet{deVriesCaswell2019b} under obligate outcrossing. A full derivation of the model and analyses is presented in \hl{Appendix A}, and all simulation code necessary to reproduce the results are available at \url{https://github.com/colin-olito/SA-Hermaphrodites-wDemography}.

Simultaneous hermaphrodites express both male and female sex functions (i.e., can be both a mother and father), and have the potential to reproduce by a combination of self- and outcross-fertilizations. Maternal outcrossing involves receiving male gametes from another individual in the population, while paternal outcrossing involves exporting male gametes to another individual and fertilizing their ovules. Self-fertilization is achieved when an individual's male gametes fertilize their own ovules. To distinguish between parameters relating to male and female function, we denote matrices or vectors relating to male sex function with a prime. 

Individuals in the model are jointly classified by life-cycle stage ($1, \ldots, \omega$), genotype ($1, \ldots, g$), and whether they were produced by self- or outcross fertilization (denoted by $S$ and $X$ superscripts, respectively). Whether individuals were produced through selfing or outcrossing is included in the individual state description because individuals produced through selfing might experience reduced survival, growth, or maturation rates as a consequence of inbreeding depression. To model inbreeding depression, we therefore keep track of how individuals were produced. Of course the level of inbreeding depression will not only be a function of whether the current individual is produced through inbreeding but also of how many generations of inbreeding have occurred (see for example, \cite{kelly1999response} and \cite{kelly2007mutation}) but keeping track of the history of inbreeding is beyond the scope of this paper. Therefore we use the commonly made assumption that individuals produced through selfing suffer a fixed level of inbreeding depression (\textbf{which reference(s) would work best here?}).

% Each genotype is characterized by a matrix of transition probabilities (including survival), and a matrix of reproductive output through male and female function. These matrices can include time variation or nonlinearities reflecting the environment or density dependence (see for example, \cite{de2020matrix}), but these will not be modelled here. Each stage contributes offspring to genotypes at the next time step according to transition matrices that are determined by the mating system and stage-structure of the population.

The population state at time $t$ is described by a population state vector, $\tilde{\mbf{n}}(t)$, which is ordered by how individuals were produced (selfing vs.~outcrossing), then by genotype, and finally by stage. For a single diallelic locus with alleles $A$ and $a$, we have three genotypes ($AA,\, Aa,\, aa$; $g = 3$), giving the population state vector:
\begin{linenomath*}
\begin{equation} \label{eq:PopStateVec}
	\tilde{\mbf{n}}(t) =  \left[
								\begin{array}{c}
									\mbf{n}^{S}_{AA}(t) \\
									\mbf{n}^{S}_{Aa}(t) \\
									\mbf{n}^{S}_{aa}(t) \\ \hline
									\mbf{n}^{X}_{AA}(t) \\
									\mbf{n}^{X}_{Aa}(t) \\
									\mbf{n}^{X}_{aa}(t) \\ 
						\end{array} \right],
\end{equation}
\end{linenomath*}

\noindent where $\mbf{n}^{S}_{i}$ and $\mbf{n}^{X}_{i}$ ($i \in \{AA,Aa,aa\}$) are the stage distribution vectors of individuals of genotype $i$ produced by self-fertilization and outcrossing, respectively. The proportional population vector is given by
\begin{linenomath*}
\begin{equation} \label{eq:propPopVec}
	\tilde{\mbf{p}}(t) = \frac{\tilde{\mbf{n}}(t)}{ \| \tilde{\mbf{n}}(t) \|},
\end{equation}
\end{linenomath*}

\noindent where $\| \cdot \|$ is the one-norm. The population vector $\tilde{\mbf{n}}(t)$ is projected forward from time $t$ to $t + 1$ by the $projection$ matrix $\tilde{\mbf{A}}(\tilde{\mbf{n}})$ such that 
\begin{linenomath*}
\begin{equation}
	\tilde{\mbf{n}}(t + 1) = \tilde{\mbf{A}}[\tilde{\mbf{n}}(t)] \, \tilde{\mbf{n}}(t)
\end{equation}
\end{linenomath*}

The population projection matrix $\tilde{\mbf{A}}$ is constructed from four sets of matrices representing the demographic and genetic processes: The matrices $\mbf{U}^{S}_{i}$ and $\mbf{U}^{X}_{i}$ contain transition and survival probabilities for each genotype, produced by selfing and outcrossing respectively. The matrices $\mbf{F}_{i}$ and $\mbf{F}^{\prime}_{i}$ contain the genotype $\times$ stage specific contribution via female and male sex-functions of genotype $i$ to the female and male gamete pools, respectively, and therefore to zygotes in the next generation. We assume that whether individuals were produced through selfing or outcrossing does not affect their fecundity or mating success, that is, we assume  $\mbf{F}^X_{i}=\mbf{F}^S_{i}=\mbf{F}_{i}$ and $\mbf{F}^{\prime X}_{i}=\mbf{F}^{\prime S}_{i}=\mbf{F}^{\prime}_{i}$. Deviations from this assumption are straightforward to incorporate but beyond the scope of the current paper. 

% Notational note: The superscript on a matrix in age$\times$stage models conventionally indicates the stage or age of the subsection of the population vector that the matrix is acting on (see for example \cite{CaswellEtAl2018}). For example, $\mbf{U}^{S}_{i}$, acts on the vector of individuals of genotype $i$ that were produced through selfing. In the following, we deviate from this convention by also using $S$ and $X$ as superscripts to indicate production of offspring through current selfing or reproduction. That is,  we will write production $\mcal{F}^S$ and $\mcal{F}^X$ to denote offspring production by self-fertilization and outcrossing. Likewise, we define the matrices $\mbf{H}^S_{j}(\tilde{\mbf{n}})$ and $\mbf{H}^X_{j}(\tilde{\mbf{n}})$ as the map of the genotypes of the parent in stage $j$ to the genotypes of their offspring produced by selfing and outcrossing, respectively. The $(k, l)$ entry of $\mbf{H}^{X}_{j}$ and $\mbf{H}^{S}_{j}$ is the probability that an offspring of a genotype $l$ mother, of stage $j$, has genotype $k$. Since we assume that how-you-were-produced does not affect how you reproduce, these matrices will act on both $\mbf{n}^{S}$ and $\mbf{n}^{X}$. 

For the purpose of this article, we assume that mating is random with respect to stage and hence that the parent-offspring map is the same for all stages (i.e., $\mbf{H}^S_{j}(\tilde{\mbf{n}}) = \mbf{H}^S(\tilde{\mbf{n}})$, and $\mbf{H}^X_{j}(\tilde{\mbf{n}}) = \mbf{H}^X(\tilde{\mbf{n}})$). The matrices $\mbf{H}^S(\tilde{\mbf{n}})$ and $\mbf{H}^X(\tilde{\mbf{n}})$ contain the population genetic processes and are presented in the next section.

%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Mating and offspring production under partial selfing}
 Outcrossing in our model proceeds similarly to a two-sex model of reproduction \citep{deVriesCaswell2019b}. That is, each individual's genotype determines both the number of ovules produced and the paternal "mating success", broadly defined. For hermaphroditic flowering plants, for example, paternal mating success could reflect pollen production, export efficiency, pollen-tube germination and growth rates, among other things \citep{LloydWebb1986, WangBarrett2020, Harder2016}. 

Since we assume that how individuals were produced does not affect their relative mating success, the frequency in the male mating population is obtained simply by summing the vectors of individuals produced through selfing and through outcrossing, that is, 
\begin{equation}
 	\mbf{p}^{\prime}=\frac{(\mbf{n}^X + \mbf{n}^S)}{\| (\mbf{n}^X + \mbf{n}^S)\|}.
 \end{equation} 
Depending on their stage and genotype at the SA locus, individuals contribute male gametes to the overall population gamete pool according to the following equation
\begin{linenomath*}
\begin{equation}
		\mbf{W}^{\prime} \mbb{F}^{\prime} \mbf{p}^{\prime}= 
		\left(
			\begin{array}{ccc}
				\mbf{1}^{\intercal}_{\omega} & \frac{1}{2} \mbf{1}^{\intercal}_{\omega} & 0 \\
				0 & \frac{1}{2} \mbf{1}^{\intercal}_{\omega} & \mbf{1}^{\intercal}_{\omega} \\
			\end{array} \right)
		\left(
			\begin{array}{ccc}
				\mbf{F}^{\prime}_{AA} & 0 & 0 \\
				0 & \mbf{F}^{\prime}_{Aa} & 0 \\
				0 & 0 & \mbf{F}^{\prime}_{aa} \\
			\end{array} \right)
		\left(
			\begin{array}{c}
				\mbf{p}^{\prime}_{AA} \\
				\mbf{p}^{\prime}_{Aa} \\
				\mbf{p}^{\prime}_{aa} \\
			\end{array} \right).
\end{equation}
\end{linenomath*}

\noindent The matrix $\mbb{F}^{\prime}$ is composed of genotype-specific fertility matrices, and operates on the the vector of genotype frequencies to give their relative contribution of each genotype to the gamete pool. The matrix $\mbf{W}^{\prime}$ converts these contributions to allele numbers. Normalizing the resulting vector gives the allele frequencies in the male gamete pool:
\begin{linenomath*}
\begin{equation} \label{eq:maleGametePool}
	\left(
		\begin{array}{c}
			q^{\prime}_{A} \\
			q^{\prime}_{a} \\
		\end{array} \right) = 
			\frac{\mbf{W}^{\prime} \mbb{F}^{\prime} \mbf{p}^{\prime}}{\| \mbf{W}^{\prime} \mbb{F}^{\prime} \mbf{p}^{\prime} \|} = 
				\frac{\mbf{W}^{\prime} \mbb{F}^{\prime} (\mbf{n}^X + \mbf{n}^S)}{\| \mbf{W}^{\prime} \mbb{F}^{\prime} (\mbf{n}^X + \mbf{n}^S)\|}
\end{equation}
\end{linenomath*}

\textcolor{red}{
I'm doubting whether to move the next paragraph to the appendices as well.  Equation \ref{eq:maleGametePool} is all we strictly need. If we keep the parent-offspring maps, we can explain in the next section how the matrices 
$\mcal{F}^S$ and $\mcal{F}^X$ have the same structure. Would that be helpful? Or would it be easier to just explain in the next section what the structure of $\mcal{F}^S$ and $\mcal{F}^X$ is? }

The parent-offspring map for outcrossing is a function of the allele frequencies in the male gamete pool, and is given by 
\begin{linenomath*}
\begin{equation} \label{eq:HX}
	\mbf{H}^X = 
			\left(
			\begin{array}{ccc}
				q^{\prime}_{A} & \frac{1}{2} q^{\prime}_{A} & 0 \\
				q^{\prime}_{a} & \frac{1}{2} & q^{\prime}_{A}  \\
				0 & \frac{1}{2} q^{\prime}_{a} & q^{\prime}_{a} \\
			\end{array} \right)
\end{equation}
\end{linenomath*}
\noindent From left to right, the columns of matrix $\mbf{H}^X$ give the genotype distribution of outcrossed offspring produced by a maternal parent of each genotype ($AA$, $Aa$, and $aa$ respectively).

Reproduction via self-fertilization differs from outcrossing. As in previous models of partial selfing, we assume that individuals produce enough male gametes to easily self-fertilize all of their ovules, and that self-fertilization involves little or no selection from external factors relative to outcrossing (e.g., \citealt{Charlesworth1978a,JordanConnallon2014,Olito2017}, but see \citealt{Tazzyman2015}). Under these assumptions, the genotype distributions of selfed offspring are determined entirely by the parental genotype and the probabilities of segregation and fertilization during and after meiosis: 
\begin{linenomath*}
\begin{equation} \label{eq:HS}
	\mbf{H}^S(\tilde{\mbf{n}}) = 
			\left(
			\begin{array}{ccc}
				1 & 1/4 & 0 \\
				0 & 1/2 & 0 \\
				0 & 1/4 & 1 \\
			\end{array} \right).
\end{equation}
\end{linenomath*}
\noindent The columns of $\mbf{H}^S(\tilde{\mbf{n}})$ again give the selfed offspring genotype distributions for parental genotypes of $AA$, $Aa$, and $aa$, respectively.


%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Population projection}
Using the component matrices described above (the survival matrices, $\mathbf{U}^X_i$, $\mathbf{U}^S_i$, the fertility and mating success matrices, $\mathbf{F}_i$, $\mathbf{F}^\prime_i$, and the parent-offspring matrices $\mbf{H}^S(\tilde{\mbf{n}})$ and $\mbf{H}^X$), we construct the population projection matrix $\tilde{\mbf{A}}[\tilde{\mbf{n}}]$ using the vec-permutation approach of \citet{CaswellEtAl2018}, see \hl{Appendix A.1} for the step by step construction of the model. The matrix that projects the eco-evolutionary dynamics is:
\begin{linenomath*}
\begin{equation} \label{eq:Atilde}
	\tilde{\mbf{A}}[\tilde{\mbf{n}}] = 
			\underbrace{\left(
			\begin{array}{c|c}
				\mcal{U}^S & \mbf{0} \\ \hline
				\mbf{0} & \mcal{U}^X \\
			\end{array} \right)}_{\tilde{\mbf{U}}} + 
			\underbrace{\left(
			\begin{array}{c|c}
				\mcal{F}^S(\tilde{\mbf{p}}) C(1 - \delta) & \mcal{F}^S(\tilde{\mbf{p}}) C(1 - \delta) \\ \hline
				\mcal{F}^X(\tilde{\mbf{p}}) (1 - C) & \mcal{F}^X(\tilde{\mbf{p}}) (1 - C)\\
			\end{array} \right)}_{\tilde{\mbf{F}}},
\end{equation}
\end{linenomath*}
where $C$ denotes the proportion of each individual's ovules that are self-fertilized (the remaining $1 - C$ are outcrossed), and  $\delta$ represents the proportion of self-fertilized zygotes that fail to develop due to inbreeding depression during early development \citep{Charlesworth1987}. In \hl{Appendix A.x}, we present a more general model of genotype-specific self-fertilization rates (after \citealt{JordanConnallon2014}). 

The blocks of the component matrices in Eq(\ref{eq:Atilde}) correspond to production of offspring by self-fertilization and outcrossing ($\mcal{F}^S$ and $\mcal{F}^X$ in $\tilde{\mbf{F}}$), and survival of selfed and outcrossed offspring ($\mcal{U}^S$ and $\mcal{U}^X$ in $\tilde{\mbf{U}}$). The survival matrix for individuals produced through selfing and outcrossing are,
\begin{linenomath*}
\begin{eqnarray} \label{eq:BlkUS}
	\mcal{U}^S  = 
		\left(
			\begin{array}{ccc}
				\mbf{U}^{S}_{AA} & \mbf{0} & \mbf{0} \\
				\mbf{0} & \mbf{U}^{S}_{Aa} & \mbf{0} \\
				\mbf{0} & \mbf{0} & \mbf{U}^{S}_{aa} \\
			\end{array} \right),\\
				\mcal{U}^X  = 
		\left(
			\begin{array}{ccc}
				\mbf{U}^{X}_{AA} & \mbf{0} & \mbf{0} \\
				\mbf{0} & \mbf{U}^{X}_{Aa} & \mbf{0} \\
				\mbf{0} & \mbf{0} & \mbf{U}^{X}_{aa} \\
			\end{array} \right).
\end{eqnarray}
\end{linenomath*}
Since individuals cannot change their genotype once they are born, the survival matrices are block diagonal.  Similarly, we construct fertility matrices for individuals produced through selfing and outcrossing, 
\begin{linenomath*}
\begin{equation} \label{eq:BlkFS}
	\mcal{F}^S = 
			\left(
			\begin{array}{ccc}
				\mbf{F}_{AA} & \frac{1}{4} \mbf{F}_{Aa} & 0 \\
				0 & \frac{1}{2} \mbf{F}_{Aa} & 0 \\
				0 & \frac{1}{4} \mbf{F}_{Aa} & \mbf{F}_{aa}\\
			\end{array} \right), 
\end{equation}
\end{linenomath*}
\noindent and 
\begin{linenomath*}
\begin{equation} \label{eq:BlkFX}
	\mcal{F}^X = 
			\left(
			\begin{array}{ccc}
				q^{\prime}_{A} \mbf{F}_{AA} & \frac{1}{2} q^{\prime}_{A} \mbf{F}_{Aa} & 0 \\ 
				q^{\prime}_{a} \mbf{F}_{AA} & \frac{1}{2} \mbf{F}_{Aa} & q^{\prime}_{A} \mbf{F}_{aa}  \\ 
				0 & \frac{1}{2} q^{\prime}_{a} \mbf{F}_{Aa} & q^{\prime}_{a} \mbf{F}_{aa}  \\
			\end{array} \right),
\end{equation}
\end{linenomath*}
\noindent where $q^{\prime}_A$ and $q^{\prime}_a$ are given by Eq(\ref{eq:maleGametePool}). Further details of the derivations of $\mcal{U}^S$, $\mcal{U}^X$ ,  $\mcal{F}^S$, and $\mcal{F}^X$ are provided in \hl{Appendix A.x}. 

The blocks of $\mcal{F}^X(\tilde{\mbf{p}})$ can be constructed and interpreted as follows: The first row block of the first column produces $AA$ offspring by outcrossing from $AA$ maternal parents. This happens when the $AA$ maternal parent receives an $A$ gamete from the male gamete pool, which happens with probability $q^{\prime}_{A}$. The other blocks can be interpreted similarly.

Combining all the component matrices yields the overall eco-evolutionary projection matrix shown in \hl{Appendix A.x}.


%%%%%%%%%%%%%%%%%%%%%%
\subsection{Sexually antagonistic selection \& inbreeding depression} \label{sec:SAsel}

We now construct and analyze a genotype $\times$ stage-classified model for a hypothetical species with intralocus sexual conflict via the two sex functions. To keep the model analytically tractable, we assume our hypothetical species has two life stages: juveniles and adults (i.e., $\omega = 2$), and that only adults are reproductively active. Suppose that there is a genetic trade-off between the sex-functions at a single bi-allelic locus such that allele $A$ is beneficial for female fertility but detrimental for male reproductive success (e.g., pollen production), and that allele $a$ has the reverse effect. 

Following convention, we parameterize the fertility component of fitness of each genotype through each sex function, $w_{i}$ and $w^{\prime}_{i}$, to be bounded by $[0,1]$, with dominance and selection coefficients $h_f, s_f$ and $h_m, s_m$ determining the decrease in fertility through each sex function relative to the most fit genotype ($AA$ has highest female fertility, $aa$ the highest male fertility; see Table \ref{tab:Fitness}). The SA locus does not affect survival and transition rates. However, the survival matrices can be used to model the fitness effects of inbreeding depression at later stages of development by allowing different stage-specific survival and transition rates for individuals produced by self-fertilization vs.~outcrossing. By contrast, the parameter $\sigma$ only affects inbreeding depression through viability of selfed ovules. With this in mind, we define survival matrices for individuals produced by selfing and outcrossing as follows:
\begin{linenomath*}
\begin{equation} \label{eq:US}
	\mbf{U}^S = \left(
					\begin{array}{cc}
						\sigma_j (1 - \delta_j) \big(1 - \gamma (1 - \delta_{\gamma}) \big) & 0 \\
						\sigma_j (1 - \delta_j) \gamma (1 - \delta_{\gamma})      & \sigma_a (1 - \delta_a)
					\end{array}
				\right) \\
\end{equation}
\end{linenomath*}
\noindent and 
\begin{linenomath*}
\begin{equation}\label{eq:UX}
	\mbf{U}^X = \left(
					\begin{array}{cc}
						\sigma_j(1 - \gamma) & 0 \\
						\sigma_j \gamma      & \sigma_a
					\end{array}
				\right)
\end{equation}
\end{linenomath*}

\noindent where $\sigma_j$ and $\sigma_a$ are the juvenile and adult stage survival rates, $\gamma$ is the maturation rate from juvenile to adult stages, and the corresponding $\delta_j$, $\delta_a$, and $\delta_{\gamma}$ terms denote the proportional decreases in stage-specific survival and transition rates due to inbreeding depression (i.e., deleterious effects of inbreeding at later life-history stages; e.g., \citealt{HarderRoutely2006}). For simplicity, we assume survival and transition rates are constant among genotypes. 

Throughout our analyses, we distinguish between early- and late-acting inbreeding depression. We quantify early-acting inbreeding depression using $\delta$, and late-acting inbreeding depression using $\delta_i$ (where $i \in \{j,a,\gamma\}$). An important difference between early- and late-acting inbreeding depression in the model is that $\delta$ does not affect $\mbf{U}^S$, the survival matrix for offspring produced by selfing (see Eq.~\ref{eq:Atilde}), while the $\delta_i$ terms directly alter the stage-specific survival and transition rates in $\mbf{U}^S$ (see Eq.~\ref{eq:US}).

The fertility matrices through female and male function are
\begin{linenomath*}
\begin{equation}\label{eq:FS}
	\mbf{F}_{i} = \left(
					\begin{array}{cc}
						0 & f w_{i} \\
						0 & 0
					\end{array}
				\right),
\end{equation}
\end{linenomath*}
\noindent and
\begin{linenomath*}
\begin{equation}\label{eq:FX}
	\mbf{F}^{\prime}_{i} = \left(
					\begin{array}{cc}
						0 & f^{\prime} w^{\prime}_{i} \\
						0 & 0
					\end{array}
				\right),
\end{equation}
\end{linenomath*}

\noindent where $f$ and $f^{\prime}$ represent adult fertilities, and $w_{i}$ and $w^{\prime}_{i}$ the genotypic relative scaling factors for female and male sex-functions (see Table \ref{tab:Fitness}). 

\begin{table}[htbp]
 \centering
 \caption{\bf Relative fertilities for Sexually Antagonistic selection ($w_{i}$)}
\label{tab:Fitness}
\begin{tabular}{lccc}
 \toprule
					&  \multicolumn{3}{c}{{\textit{Genotype}}} \\ 
\cline{2-4}
					& $AA$			& $Aa$ 					& $aa$ 		\\ \hline
Female function ($w_{i}$):	& $1$		& $1 - h_f s_f$	& $1 - s_f$ \\	
Male function ($w^{\prime}_{i}$):		& $1 - s_m$& $1 - h_m s_m$	& $1$ 		\\	
\hline
\end{tabular}
\end{table}

Iterating the projection matrix, Eq(\ref{eq:Atilde}) with the above demographic matrices, given an initial population state vector, Eq(\ref{eq:PopStateVec}), allows numerical simulation of the eco-evolutionary dynamics for selection operating on any of the sex-function, or stage-specific demographic parameters. As we outline below, we use numerical techniques together with mathematical analyses to study the conditions for the maintenance of SA polymorphisms, and the demographic fate of the populations (i.e., positive growth, or extinction).

Unless stated otherwise, we use the following parameter values for the demographic rates in the model: $\sigma_j = \sigma_a = 0.6$ and $\gamma = 0.05$, which are similar to those used in \citet{deVriesCaswell2019b}. Our parameters of interest include fertility, $f$, the inbreeding depression parameters, $\delta$, $\delta_j$, $\delta_a$, $\delta_{\gamma}$, and the selection parameters $h_f$, $s_f$, $h_m$, and $s_m$, which are given different values for each analyses as described in the figure captions.

%%%%%%%%%%%%%%%%%%%%%%
\subsection{Analyses} \label{subsec:analyses}

Diverse eco-evolutionary outcomes are possible in the model, including fixation of either allele, balanced polymorphism, population growth or extinction, and even evolutionary rescue and suicide (e.g., see \citealt{CaswellEtAl2018,deVriesCaswell2019a,deVriesCaswell2019b}). We focus on identifying parameter conditions where two criteria are satisfied: ({\itshape i}) SA polymorphism is maintained under balancing selection and ({\itshape ii}) the intrinsic population growth rate at equilibrium is positive; a situation that we refer to as a 'demographically viable SA polymorphism'.

We identify conditions where SA polymorphism is 'protected' by evaluating the stability of populations initially fixed for either SA allele to invasion by the other (i.e., stability was assessed at the boundary equilibrium genotype frequencies of $\hat{\mbf{p}}_{AA} = 1$ and $\hat{\mbf{p}}_{aa} = 1$; \citealt{Levene1953, Prout1968, deVriesCaswell2019b}). The formal conditions for a protected polymorphism are determined by linearizing the model in the vicinity of the boundary equilibria ($\hat{\mbf{p}}$), and evaluating the magnitude of the largest eigenvalue of the Jacobian matrix of the linearization. A full derivation of the Jacobian and details of the invasion analysis are provided in \hl{Online Appendix X.x}.

%denoted by $\tilde{\zeta}_{AA}$ and $\tilde{\zeta}_{aa}$ for the eigenvalue of the Jacobian evaluated at the $AA$ and $aa$ boundary, respectively. For our model, the Jacobian, $\mbf{M}$, has an upper block triangular form, and we can therefore express the stability of $\hat{\mbf{p}}$ in terms of the leading eigenvalue of the diagonal block corresponding to changes in heterozygote frequency ($\mbf{M}_{22}$; e.g., see \citealt{OttoDay2007,deVriesCaswell2019b}). For a polymorphism to be 'protected', both boundary equilibria must be unstable (i.e., $\tilde{\zeta}_{AA} > 1$ and $\tilde{\zeta}_{aa} > 1$), which requires that
% \begin{linenomath*}
% \begin{equation}\label{eq:PolyCond}
% 	\rho(\mbf{M}_{22}) > \lambda_{i}
% \end{equation}
% \end{linenomath*}

% \noindent is satisfied, where $\rho(\cdot)$ represents the spectral radius,  $\rho(\mbf{M}_{22})$ gives the ergodic growth rate of perturbations of the boundary equilibrium according to the linear approximation, and $\lambda_i$ is the intrinsic growth rate for the homozygous genotype being invaded at each boundary ($i \in (AA,aa)$; see Box.~1). 


We used numerical simulation to determine whether a protected SA polymorphism was also demographically viable. Specifically, for each boundary equilibrium we introduced the rare allele at low initial frequency, and iterated Eq(\ref{eq:Atilde}) until the population had reached demographic and genotypic equilibrium. In our model, the population state vector will grow or shrink exponentially after converging to stable population structure and genotypic frequencies (see \citealt{Caswell2001}), and the intrinsic population growth rate after convergence, $\lambda$, can be calculated as $\tilde{\mbf{n}}(t)/\tilde{\mbf{n}}(t-1)$. We note, however, that if the ecological component of the model is non-linear, more exotic dynamics are possible \citep{de2020matrix}.

% Unfortunately, closed-form solutions for the invasion conditions for each allele and the equilibrium frequencies under different selection scenarios are analytically intractable except for restricted conditions (see \hl{Online Appendix X}). We therefore used numerical techniques to identify invasion and extinction thresholds, which could then be used to define regions of demographically viable polymorphism across $s_f \times s_m$ parameter space. 
Because single-locus selection coefficients are generally weak (i.e., $s < 0.1$; e.g., \citealt{Eyre-WalkerKeightly2007}) and strongly skewed, we limit our analyses to coefficients within $0 < s_f,s_m \leq 0.15$, unless stated otherwise. For a given parameter set, we numerically evaluated the largest eigenvalue of the Jacobian evaluated at both boundaries, and the population growth rate of the equilibrium obtained after perturbation of the boundary, $\lambda$. To determine the viability threshold (where the population growth rate equals one) and the boundary stability thresholds (where the largest eigenvalue of the Jacobian crosses one), we start at the outer boundaries of selection parameter space, and titrated inward. The proportions of total $s_f \times s_m$ parameter space where SA polymorphism and/or extinction occurred were calculated by numerically integrating the appropriate regions defined by the invasion and extinction thresholds

For analyses of the demographic effects of inbreeding depression, we made two main simplifying assumptions: First, we acknowledge that if inbreeding depression is caused primarily by recessive deleterious mutations (as suggested by empirical data), it should covary negatively with the population selfing rate due to purging (\citealt{Charlesworth2009}; though we note that other processes could give rise to this pattern, e.g., \citealt{CrnokrakBarrett2002, Charlesworth2009,HedrickGarcia-Dorado2016}). Following \citet{Olito2019}, we incorporate such negative covariance by constraining the inbreeding depression terms in the model ($\delta$ and $\delta_i$, where $i \in \{J,A,\gamma\}$) to follow a simple declining function of the selfing rate: $\delta = \delta^{\ast} (1 - b (1 - L))$, where $\delta^{\ast}$ is the hypothetical severity of inbreeding depression for a completely outcrossing population, $b$ is a shape parameter determining how far $\delta$ will decline under complete selfing (when $C = 1$), and $L$ is describes the expected deleterious mutation load as a function of the selfing rate $C$, due to recessive deleterious mutations. The function $L$ includes an additional shape parameter, $a$, which determines the curvature of the overall function for $\delta$ (see Appendix E in \citealt{Olito2019} for additional details). We set $\delta^{\ast} = 0.8$, $b = 0.5$, and $a = 0.2$ for all analyses, values chosen to be consistent with empirical estimates of inbreeding depression (e.g., fig.~2 in \citealt{HusbandSchemske1996}). Second, we explore the effects of individual inbreeding depression terms in isolation. That is, we assume that only one of the terms $\delta$ and $\delta_i$ (where $i \in \{J,A,\gamma\}$) can be non-zero at a time. 




%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%
\section{Results}\label{sec:Results}
%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%
\subsection{Polymorphism and Extinction}\label{subsec:PolyExt}


%%%%%%%%%%%%%%%%%%%%%
% Figure 1
 \begin{figure}[htbp]
 \centering
 \includegraphics[width=\linewidth]{../output/figs/extinctionThresholdsFig.pdf}
 \caption{\footnotesize{Illustration of parameter space for SA polymorphism and extinction thresholds predicted by the mendelian stage-structured matrix model. Balanced SA polymorphisms can be maintained in the funnel-shaped region between the invasion conditions for each SA allele (dark solid lines). However, for some parameter conditions, populations will ultimately go extinct (red shaded regions) due to reduced female fitness resulting from the male-beneficial/female-deleterious allele that is either segregating as a balanced polymorphism (inside the funnel), or becomes fixed (area below the funnel). "Demographically viable polymorphic parameter space" corresponds to the area inside the funnel that is also to the left of the extinction threshold for a given fertility value. Results are shown for three different population selfing rates ($C = \{0,\,1/4,\,1/2\}$), and two dominance scenarios (additivity, where $h = 1/2$, and dominance reversal, where $h = 1/4$); extinction thresholds are illustrated for three different values of female fecundity ($f$ values annotated on each panel).}}
 \label{fig:extThresholds}
 \end{figure}
%%%%%%%%%%%%%%%%%%%%%

We begin with a simple illustration of demographically viable polymorphic parameter space in the absence of inbreeding depression (i.e., $\delta_i = 0$). Invasion conditions for SA alleles in the evolutionary demographic model closely match the predictions from population genetic models (see \hl{Online Appendix X} \citealt{Kidwell1977, JordanConnallon2014, Olito2017}). In particular, the demographic model recovers the classic "funnel-shaped" region of polymorphic $s_f \times s_m$ parameter space. The effects of the population selfing rate ($C$) and dominance ($h$) on SA polymorphism are also similar: self-fertilization ({\itshape i}) skews the SA polymorphic space towards regions with weaker selection through female-function and reduces the total area of polymorphic parameter space; and ({\itshape ii}) dominance reversals (where deleterious SA fitness effects are partially recessive in each sex; $h = 1/4$) are much more permissive of SA polymorphism \citep{JordanConnallon2014, Olito2017}. 

However, a key prediction from the evolutionary demographic model is that large fractions of SA polymorphic parameter space can be demographically inviable (Fig.~\ref{fig:extThresholds}). The location of the extinction threshold, where the population intrinsic growth rate $\lambda = 1$, is primarily determined by the fertility parameter ($f$) but is also influenced by the population selfing rate ($C$) and dominance of the SA alleles ($h$). When $f$ is large, the population can sustain higher mortality caused by stronger selection through both male and female fitness components before extinction occurs (the extinction threshold shifts to higher values of $s_f$ and $s_m$). 

The demographic effects of SA polymorphism are driven by the fitness effects of segregating male-beneficial/female-deleterious alleles because they directly influence individuals' fecundity. For the fertility parameter values we explored, all extinctions occured in regions where either the female-deleterious allele segregates at intermediate frequency (i.e., where SA polymorphism is maintained), or goes to fixation (fig.~\ref{fig:extThresholds}). The SA polymorphisms that remained viable at lower fertilitiy values corresponded to regions where the female-deleterious allele is predicted to segregate at low frequencies, or in regions with weak selection through both sex functions. 


In populations with high fertility (larger $f$), the proportion of demographically viable polymorphic parameter space converges on the predictions for total SA polymorphic space in population genetic models (fig.~\ref{fig:polySpace}). In obligately outcrossing populations (including dioecious/gonochoristic populations; where $C = 0$), lower fertility can result in a significant reduction of demographically viable polymorphic parameter space. However, the effect is weaker in populations with intermediate selfing rates ($C > 0$) because self-fertilization protects a fraction ($C$) of the ovules of individuals homozygous for the female beneficial allele from exposure to the allele that is deleterious for females. In other words, self-fertilization leads to less selective death. This allows partially selfing populations to remain viable under selection intensities that would cause extinction in an outcrossing population. Interestingly, when fertility is low enough that some polymorphic parameter space is demographically inviable, populations with intermediate selfing rates have the greatest proportion of demographically viable parameter space (fig.~\ref{fig:polySpace}, med.~and low.~fertility values). This effect emerges from the combination of protection from selective death and reduced total polymorphic parameter space caused by selfing. 

% Figure 2
 \begin{figure}[htbp]
 \centering
 %\includegraphics[scale=0.75]{../output/figs/polymorphicSpaceTitrate.pdf}
 \caption{\footnotesize{Proportion of demographically viable parameter space (out of total $s_f \times s_m$ space with $\max(s) = 0.15$) in the absence of inbreeding depression (i.e., assuming $\delta = \delta_i = 0$, where $i \in \{j,a,\gamma\}$), plotted as a function of the population selfing rate. Results are shown for three fertility values corresponding to low, medium, and high fertility (blue, green, and red points respectively) under additive ($h = 1/2$; panel A), and partially recessive ($h = 1/4$; panel B) SA fitness effects. Each point was calculated by numerical integration of the corresponding SA invasion conditions and extinction threshold predicted by the mendelian matrix model (see \hyperref[subsec:analyses]{Analyses} section), while solid lines were produced by numerically integrating the analytic expressions for the single-locus invasion conditions from the population genetic models of \citet{JordanConnallon2014} and \citet{Olito2017} (solid black lines).}} 
 \label{fig:polySpace}
 \end{figure}



%%%%%%%%%%%%%%%%%%%%%%
\subsection{Demographic effects of inbreeding depression} \label{subsec:InbreedEffects}

Unlike previous population genetic models which assumed constant population sizes \citep{JordanConnallon2014,Olito2017}, mortality caused by inbreeding depression can strongly influence population persistence in our evolutionary demographic model. Populations with high fertility rates can sustain positive population growth rates despite this higher mortality. This leads to a greater proportion of SA polymorphic parameter space to be demographically viable, with the demographic model predictions converging on those from population genetic models in high-fertility populations (fig.~\ref{fig:deltaPolySpace}). For populations with lower fertility rates, the proportion of demographically viable polymorphic parameter space matches the population genetic model predictions with the selfing rate is low; however, as the selfing rate increases, demographic viability eventually crashes when the population can no longer sustain the concomitant increase in mortality due to inbreeding depression (fig.~\ref{fig:deltaPolySpace}). 

Mortality from inbreeding depression reduces the effect of protection from selective death that individuals gain through self fertilization. In contrast to our earlier results (see \hyperref[subsec:PolyExt]{Polymorphism and Extinction}), predominantly outcrossing populations are predicted to have the highest proportion of demographically viable polymorphic space, when inbreeding depression is taken into acount (compare fig.~\ref{fig:polySpace} with fig.~\ref{fig:deltaPolySpace}). Regardless of the life-history stage at which inbreeding depression affects survival, populations with intermediate to high selfing rates are unlikely to harbour SA polymorphism unless they can "afford" the resulting loss of self-fertilized ovules/offspring. \textcolor{red}{How about the flipside: Populations with intermediate to high selfing rates and growth rates around one are vulnerable to extinction if a sexually antagonistic allele invades the population.  }


% Figure 3
 \begin{figure}[htbp]
 \centering
 %\includegraphics[scale=0.75]{../output/figs/deltaPolymorphicSpaceTitrate.pdf}
 \caption{\footnotesize{Effects of early- and late-acting inbreeding depression on the proportion of demographically viable parameter space (out of total $s_f \times s_m$ space with $\max(s) = 0.15$), plotted as a function of the population selfing rate. In all plots, the strength of inbreeding depression decreases as the selfing rate goes up following a simple model of purging recessive deleterious mutations (see \hl{Appendix X.x} for details). Only single inbreeding depression terms ($\delta$ and $\delta_i$, where $i \in \{j,a,\gamma\}$, indicated in the legend) are allowed to vary at one time (all others are set to $0$). Results are shown for three fertility values ($f = \{6.5,\,7.5,\,8.5\}$) under additive ($h = 1/2$; panel A) and partially recessive ($h = 1/4$; panel B) SA fitness effects. Each point was calculated by numerically integrating the corresponding SA invasion conditions and extinction threshold predicted by the model (see \hyperref[subsec:analyses]{Analyses} section), while solid lines were produced by numerically integrating the single-locus invasion conditions from the population genetic models of \citet{JordanConnallon2014} and \citet{Olito2017} (solid black lines).}} 
 \label{fig:deltaPolySpace}
 \end{figure}


However, when during the life-cycle inbreeding depression manifests does influence the threshold selfing rate at which demographically viable polymorphic parameter space crashes. Our results indicate that population viability was most sensitive to inbreeding depression affecting juvenile survival rates ($\delta_j$; fig.~\ref{fig:deltaPolySpace}, dark blue points), while early-acting inbreeding depression ($\delta$, ovule abortion shortly after fertilization) had a similar effect on population viability as late-acting inbreeding depression affecting adult survial ($\delta_a$) and juvenile-to-adult transition rates ($\delta_{\gamma}$). Inbreeding effects on juvenile survival had the strongest effect on population viability because on average individuals will spend multiple time steps in the juvenile stage before they mature. At each time step in the juvenile stage, individuals have a probability $\gamma$ to mature and a probability $\sigma_j(1-\delta_j)$ to survive. Inbreeding depression at the juvenile stage therefore makes it harder to survive long enough to mature. Although early-acting inbreeding depression ($\delta$) actually manifests earlier in the life-cycle than juvenile survival, it acts only once by influencing the total number of self-fertilized zygotes that become juveniles.


\section{Case study: {\itshape M. guttatus}}

%\subsection{Demographic and fitness data for {\itshape M. guttatus}} \label{subsec:MguttMethods}

As an illustrative example of how our model can be used to explore whether demographic rates observed in natural populations appear likely to support balanced SA polymorphisms, we parameterized our model using empirically estimated demographic rates and fitness data for natural populations of the hermaphroditic flowering plant {\itshape Mimulus guttatus} (Scrophulariaceae; now known as {\itshape Erythranthe guttata}). {\itshape M.~guttatus} is an herbaceous, self-compatible wildflower native to western North America that exhibits remarkable among-population variation in numerous life-history and reproductive traits including selfing rates, inbreeding depression, floral morphology, and annual-to-perennial life-history \citep[e.g.,][]{RitlandGanders1987, Ritland1990, Willis1993, Willis1999a, Willis1999b, WuWillis2008}. Moreover, detailed demographic studies have been conducted on multiple populations of {\itshape M.~guttatus}, with demographic matrix data available on the public demographic database COMPADRE \citep{CompadreDB2020}. Below, we briefly outline how we parameterized our model using the available data; full details are provided in \hl{Appendix B.x}.

\subsection{Demographic and fitness data for {\itshape M. guttatus}} \label{subsec:MguttMethods}
We used extensive demographic data reported in a large-scale study of local adaptation using experimental populations of {\itshape M.~guttatus} in Stanislaus National Forest (California, USA) in 2012 and 2013 \citep{PetersonEtAl2016}. We leverage their common-garden experimental design to focus on a comparison of demographically viable polymorphic parameter space for two experimental populations with contrasting demographic rates. The first was a locally adapted 'Eagle Meadows' population (data from 2012), while the second group was an experimental population composed of multiple non-locally adapted 'low-elevation perennials' (data from 2013). The vital rate estimates for the Eagle Meadows population are as follows: seed bank survival ($D = 0.534$), seed germination rate ($G = 0.469$), flower production ($F = 0.64$), ovules per flower ($O = 614$), seedling recruits proportional to clonal rosette recruits ($A = 6.7 \times 10^{-4}$), overwinter survival ($S = 0.179$), rosette production ($R = 8.71$). The corresponding estimates for the low-elevation perrenials are: $D = 0.534$, $G = 0.652$, \textcolor{red}{$F = 0.4.09$ Should this be 0.409 or 4.09? }, $O = 494$, $A = 6.7e-4$), $S = 0$, and $R = 0$ (see corrected Tables.~1 and S2 in \citealt{PetersonEtAl2017}). The same estimates for $D$ and $A$ were used for all populations. The resulting transition matrices for this population involved three life-history stages ($\omega = 3$; seed, seedling, and rosette), and individual elements of the transition matrix ($\tilde{\mbf{A}}$) were calculated as products of the above rates (see Matrix $1$ in \citealt{PetersonEtAl2016}).

Estimates of selfing rates and inbreeding depression were not available for the same experimental populations, but are available for a variety of other western USA {\itshape M.~guttatus} populations. Selfing rate estimates vary in magnitude from near complete outcrossing to predominant selfing ($C \approx 0$ to $0.75$; \citealt{RitlandGanders1987, Ritland1990, Willis1999b}). Estimates of inbreeding depression at several of the life-history stages/fitness components that were included in the data of \citet{PetersonEtAl2016} are available for two intensively studied populations in the Cascade Mountains of Oregon (Iron Mountain and Cone Peak) \citet{Willis1993, Willis1999a, Willis1999b}. Using the data provided in \citet{Willis1993}, we estimated the proportional decrease due to inbreeding depression in seed germination rate ($\delta_{G} = 0.085$), flower number ($\delta_{F} = 0.2$), and overwinter survival ($\delta_{S} = 0.38$). The largest field-estimated selfing rate for this same Iron Mountain population was $C = 0.29$ \citep{Willis1993}.

Using these combined demographic rates, selfing rate, and inbreeding depression estimates, we constructed a corresponding stage $\times$ genotype mendelian matrix model with a single SA locus affecting female and male fertility (as described above). Using this empirically parameterized model, we are able to make predictions about the scope for demographically viable SA polymorphism at a single locus in hypothetical populations with the same demographic rates as observed in \citet{PetersonEtAl2016}, for a range of selfing and inbreeding depression rates observed in other natural populations. We stress, however, that these illustrative predictions are not explicit predictions of the likelihood of SA polymorphism in any specific population.

Interestingly, a polymorphic chromosomal inversion (inv6) with apparently SA fitness effects has been identified in the Iron Mountain population of {\itshape M.~guttatus} \citep{LeeKelly2015}. inv6 segregates at moderate frequency (about $8\%$), and carriers suffer $\approx 30 \%$ loss in pollen viability, but also increased flower production which varies among years. The genetic basis of the fitness effects caused by inv6 are not yet known, but the net result is a "supergene" with strong fitness effects that segregates as a single diallelic locus. We estimated dominance and selection coefficients for the effect of inv6 on pollen viability and flower production from the data reported in \citet{LeeKelly2015}. Across four mapping populations, the effect of inv6 on pollen viability was partially recessive on average ($h_m \approx 0.35$). Given this average dominance, the selection coefficient in the field is approximately $s_m = 0.88$. It was not possible to estimate the average dominance of the effect of inv6 on flower production (inv6 homozygotes were too rare in the field), so we conservatively assume that it was additive ($h_f = 0.5$). Based on this assumption, the selection coefficient inv6 on flower production in the field was $s_f = 0.384$ in 2012, and $s_f = 0.232$ in 2013. As a final proof-of-concept test of our empirically parameterized model \citep[e.g.,][]{Servedio2014}, we asked whether, given the above biologically grounded fitness effect estimates, inv6 appears to fall in demographically viable SA polymorphic parameter space, as might be expected given its observed frequency in the Iron Mountain population.


%%%%%%%%%%%%%%%%%%%%%%
\subsection{Polymorphism in M.~guttatus} \label{sec:Mgutt}

The Eagle Meadows and Low-Elevation Perennial populations of \citet{PetersonEtAl2016} had contrasting demographic rates that strongly affected the scope for demographically viable SA polymorphism. The Eagle Meadows population was locally adapted with a very high intrinsic growth rate ($\lambda \approx 1.7$). This growth rate was high enough that all $s_f \times s_m$ selection parameter space (where $s_f, s_m \in (0,1]$) remained demographically viable, regardless of the selfing rate ($C = 0,0.29$), or the presence of inbreeding depression (fig.~\ref{fig:mimulusFig}A). In contrast, the Low-Elevation Perennial population had a much lower intrinsic growth rate, although it was still positive ($\lambda \approx 1.08$). Due to the slower growth rate, not all of the $s_f \times s_m$ selection parameter space was demographically viable. Extinction thresholds appeared under complete outcrossing ($C = 0$) and partial selfing with inbreeding depression ($C = 0.29$), but not when inbreeding depression was omitted (fig.~\ref{fig:mimulusFig}B). The different demographic rates from these two populations also resulted in slightly different invasion conditions when inbreeding depression was taken into account (compare dotted lines in fig/~\ref{fig:mimulusFig}A and B).

Interestingly, the polymorphic inversion inv6 falls very near the lower edge of polymorphic parameter space for these hypothetical populations, but is not always predicted to be demographically viable. When using demographic rates for either the Eagle Meadows or Low-Elevation Perennial populations, inv6 falls within polymorphic space when the selfing rate is at the higher end of empirical estimates for the Iron Mountain population in which inv6 has been documented ($C = 0.29$), but only when inbreeding depression is not taken into account (fig.~\ref{fig:mimulusFig} dashed lines). When including empirical estimates of inbreeding depression, the population effective selfing rate is reduced, the polymorphic parameter space shifts upwards, and inv6 no longer falls within the polymorphic space. When using vital rates for the locally adapted Eagle Meadows population, the model predicts that an allele (or supergene) with the same fitness effects as inv6 will go extinct. In contrast, when using the demographic rates for Low-Elevation Perennials, an allele/supergene with these fitness effects falls within demographically inviable parameter space. That is, the model predicts that SA alleles with selection coefficients of similar magnitude to inv6 will inflict too high of a demographic cost, and are unlikely to be observed in populations with lower intrinsic growth rates like those observed in the Low-Elevation Perennial population.


% Figure 4
\begin{figure}[htbp]
 \centering
 %\includegraphics[scale=0.75]{../output/figs/mimulusinv6Fig.pdf}
 \caption{\footnotesize{Illustration of model predictions using empirically estimated demographic rates for {\itshape M.~guttatus}. Results are shown for two hypothetical populations using demographic rates for locally adapted (Eagle Meadows; panel A), and non-local (Low-Elevation Perennial; panel B) populations reported by \citet{PetersonEtAl2016}. Invasion conditions and extinction thresholds were calculated using the dominance coefficients estimated for inv6 ($h_f = 0.5, h_m = 0.35$; from data in \citealt{LeeKelly2015}) for three parameter conditions: obligate outcrossing (solid black lines), partial selfing using the highest field-estimate for the Iron Mountain population of \citet{Willis1993} (dashed lines), and partial selfing with inbreeding depression (dotted lines; using $\delta_i$ estimates for Iron Mountain by \citealt{Willis1993}). The location of inv6 is also shown on both plots, using estimated selection coefficients. Note that in panel A the predictions for partial selfing with inbreeding depression (dotted lines) fall extremely close to the predictions for obligate outcrossing (solid lines).}} 
 \label{fig:mimulusFig}
 \end{figure}


\hl{[SPECULATION!!!]} Finally, the location of inv6 in selection parameter space is interesting itself because in this region an SA allele with net female-beneficial fitness effects similar to inv6 are predicted to either go extinct, or segregate at relatively low frequency, depending on the population selfing-rate and severity of inbreeding depression. Although we cannot make specific predictions for inv6 in the Iron Mountain population in which it was observed, it is interesting that the estimated fitness effects place inv6 roughly in parameter space where segregation at low to moderate frequencies is not surprising. This rough concordance between theory and data is clearly speculative, but to our knowledge. 




%%%%%%%%%%%%%%%%%%%%%%%%
\section{Discussion}
%%%%%%%%%%%%%%%%%%%%%%%%
\textit{Summary} \newline
In this paper we developed a stage-structured mendelian matrix model that accommodates selection through male and female sex-functions in populations of simultaneous hermaphrodites with mixed mating systems. Using the model, we jointly analyzed the evolutionary and demographic consequences of SA selection in obligately outcrossing (equivalent to dioecious) and partially selfing hermaphrodite populations. We focused our analyses on identifying the parameter conditions under which SA polymorphisms are maintained by balancing selection {\itshape and} the population is also demographically viable (i.e., has a positive intrinsic growth rate). We also looked at the effects of inbreeding depression manifesting in different life-history stages, exploring the fitness consequences of early- vs.~late-acting inbreeding depression on the model predictions. Finally, we gave an illustrative example of the potential for SA polymorphism in real populations by parameterizing our model with empirically estimated demographic rates for the hermaphroditic flowering plant {\itshape Mimulus guttatus} (now {\itshape Erythranthe guttata}).

A key finding of our study is that when the intrinsic population growth rate is close to $1$, the deleterious effects of segregating male-beneficial SA alleles on female fecundity can result in extinction over much of the parameter space for SA polymorphism. Furthermore, this demographically viable polymorphic parameter space is often biased towards alleles with stronger selection through the female- than male sex function. Self-fertilization can alleviate the demographic costs of balanced SA polymorphisms under some conditions, however, the concommitant effects of inbreeding depression generally exacerbate them in populations with mixed-mating systems. Despite these demographic pitfalls, our example using {\itshape M.~guttatus} appears to show that demographic rates observed in some real populations are capable of sustaining large regions of viable SA polymorphic space. Overall, our findings provide a more nuanced picture of the nature of SA genetic variation that we should expect to find in natural populations, where the fate of SA alleles and the populations harboring them is determined jointly by evolutionary and demographic processes.

\textit{Extensions} \newline
By combining the tools of demography and those of population genetics, new questions can be answered with the framework we presented here. For example. many interesting extensions of our paper towards more complicated life histories are possible. For the sake of simplicity, we used a basic life cycle with just two stages for most of this paper, adults and juveniles (the \textit{Mimulus} example has 3 stages). By including more age classes, we can ask whether the scope for SA to maintain polymorphisms is affected by whether a species exhibits positive, or negative senescence \citep{jones2014diversity}? That is, does the shape of the survival curve affect the area of demographically viable polymorphic parameter space, and how does this interact with the age at which the sexually antagonistic allele is expressed? What if the allele not only affects male and female sex functions but also affects survival of individuals? And last but not least, how would the scope for SA to maintain polymorphisms be affected by density-dependence acting on different life history stages? 

We made the simplifying assumption that only how individuals themselves were produced affects the level of inbreeding depression they experience. In reality, the history of inbreeding in the entire lineage will influence the level of inbreeding depression experienced by an individual. This could be included in our framework by expanding the individual state space from produced via outcrossing or selfing towards produced via produced via outcrossing or first generation selfing, second generation selfing, etc, as in \cite{kelly1999response} and \cite{kelly2007mutation}. 


\textit{Conclusion} \newline
Despite the recent interest in eco-evolutionary dynamics, the demographic consequences of intralocus sexual antagonism have rarely been considered (but see \cite{harts2014demography}). In contrast, models of the population dynamical consequences of interlocus sexual conflict are more common, see for example \cite{tanaka1996sexual} and \cite{martinez2017sexual}. \hl{There's probably other relevant papers here, check!} In this paper we showed that including basic demography can have a significant impact on traditional population genetic results. For example,  Figure \ref{fig:extThresholds} shows that large fractions of polymorphic parameter space could lead to population extinction. This inviable fraction of parameter space is especially large for populations with growth rates already close to one (ignoring any potential compensatory density-dependent effects). 

Demographic models connect individual level traits to population level consequences. Through their focus on individuals, demographic models are ideal for linking to experimental or field data, such as capture-mark-recapture data. Doing so allows the field of population genetics to move from fitness as an abstract scalar metric towards the fitness of an entire life cycle obtained by combining observed rates of age- or stage-specific survival and fecundity rates.  



 




%%%%%%%%%%%%%%%%%%%%%%%%
%\section{Conclusion}
%%%%%%%%%%%%%%%%%%%%%%%%



%%%%%%%%%%%%%%%%%%%%%
% Acknowledgments
%%%%%%%%%%%%%%%%%%%%%

%\section{Acknowledgments}

% ... We are particularly grateful to the European Society for Evolutionary Biology (ESEB) for funding the Special Topics Network workshops âLinking local adaptation with the evolution of sex differencesâ, and to the participants, all of whom made this study possible. We would also like to thank .... This work was supported by a Wenner-Gren Foundation postdoctoral stipend to C.O., and [Lotte's funding] to C.dV..

\newpage{}

\section{Appendix A: Construction of the population projection matrix}

\subsubsection*{Population projection}

Using the component matrices described above, the matrix $\tilde{\mbf{A}}[\tilde{\mbf{n}}]$ that projects the eco-evolutionary dynamics is:
\begin{linenomath*}
\begin{equation} \label{eq:Atilde_appendix}
	\tilde{\mbf{A}}[\tilde{\mbf{n}}] = 
			\underbrace{\left(
			\begin{array}{c|c}
				\mcal{U}^S & \mbf{0} \\ \hline
				\mbf{0} & \mcal{U}^X \\
			\end{array} \right)}_{\tilde{\mbf{U}}} + 
			\underbrace{\left(
			\begin{array}{c|c}
				\mcal{F}^S(\tilde{\mbf{p}}) C(1 - \delta) & \mcal{F}^S(\tilde{\mbf{p}}) C(1 - \delta) \\ \hline
				\mcal{F}^X(\tilde{\mbf{p}}) (1 - C) & \mcal{F}^X(\tilde{\mbf{p}}) (1 - C)\\
			\end{array} \right)}_{\tilde{\mbf{F}}}.
\end{equation}
\end{linenomath*}

\noindent The blocks of the component matrices in Eq(\ref{eq:Atilde}) correspond to production of offspring by self-fertilization and outcrossing ($\mcal{F}^S$ and $\mcal{F}^X$ in $\tilde{\mbf{F}}$), and survival of selfed and outcrossed offspring ($\mcal{U}^S$ and $\mcal{U}^X$ in $\tilde{\mbf{U}}$). 

We construct $\tilde{\mbf{A}}[\tilde{\mbf{n}}]$ using the vec-permutation approach of \citet{CaswellEtAl2018}, see \hl{Appendix A.1} for the step by step construction of the model. Briefly, we construct a set of block-diagonal matrices: e.g.,
\begin{linenomath*}
\begin{equation} \label{eq:BlkUS_appendix}
	\mbb{U}^S = 
		\left(
			\begin{array}{ccc}
				\mbf{U}^{S}_{AA} & \mbf{0} & \mbf{0} \\
				\mbf{0} & \mbf{U}^{S}_{Aa} & \mbf{0} \\
				\mbf{0} & \mbf{0} & \mbf{U}^{S}_{aa} \\
			\end{array} \right),
\end{equation}
\end{linenomath*}

\noindent and corresponding matrices $\mbb{U}^X$, $\mbb{F}^S$, $\mbb{F}^X$. Since individuals cannot change their genotype once they are born, the survival matrices are block diagonal, and $\mcal{U}^S = \mbb{U}^S$ and $\mcal{U}^X = \mbb{U}^X$. 

Similarly, we also construct block-diagonal matrices
\begin{linenomath*}
\begin{eqnarray}
	\mathbb{H}^X(\tilde{\bo p}) = \bo I_\om \kron \bo H^X(\tilde{\bo p}),\\
	\mathbb{H}^S(\tilde{\bo p}) = \bo I_\om \kron \bo H^S(\tilde{\bo p}).
\end{eqnarray}
\end{linenomath*}
The corresponding fertility matrices for self-fertilization and outcrossing are defined as
\begin{linenomath*}
\begin{equation} \label{eq:BlkFS_appendix}
	\mcal{F}^S = \mbf{K}^{\intercal} \mbb{H}^S(\tilde{\mbf{p}}) \mbf{K} \mbb{F}^S = 
			\left(
			\begin{array}{ccc}
				\mbf{F}_{AA} & \frac{1}{4} \mbf{F}_{Aa} & 0 \\
				0 & \frac{1}{2} \mbf{F}_{Aa} & 0 \\
				0 & \frac{1}{4} \mbf{F}_{Aa} & \mbf{F}_{aa}\\
			\end{array} \right), 
\end{equation}
\end{linenomath*}
\noindent and 
\begin{linenomath*}
\begin{equation} \label{eq:BlkFX_appendix}
	\mcal{F}^X = \mbf{K}^{\intercal} \mbb{H}^X(\tilde{\mbf{p}}) \mbf{K} \mbb{F}^X = 
			\left(
			\begin{array}{ccc}
				q^{\prime}_{A} \mbf{F}_{AA} & \frac{1}{2} q^{\prime}_{A} \mbf{F}_{Aa} & 0 \\ 
				q^{\prime}_{a} \mbf{F}_{AA} & \frac{1}{2} \mbf{F}_{Aa} & q^{\prime}_{A} \mbf{F}_{aa}  \\ 
				0 & \frac{1}{2} q^{\prime}_{a} \mbf{F}_{Aa} & q^{\prime}_{a} \mbf{F}_{aa}  \\
			\end{array} \right),
\end{equation}
\end{linenomath*}

\noindent where $q^{\prime}_A$ and $q^{\prime}_a$ are given by Eq(\ref{eq:maleGametePool}), and $\mbf{K}$ is the vec-permutation matrix \citep{HendersonSearle1981}. Further details of the derivations of $\mcal{F}^S$ and $\mcal{F}^X$ are provided in \hl{Appendix A.x}. 

The blocks of $\mcal{F}^X(\tilde{\mbf{p}})$ can be constructed and interpreted as follows: The first row block of the first column produces $AA$ offspring by outcrossing from $AA$ maternal parents. This happens when the $AA$ maternal parent receives an $A$ gamete from the male gamete pool, which happens with probability $q^{\prime}_{A}$. The other blocks can be interpreted similarly.

Combining all the component matrices yields the overall eco-evolutionary projection matrix shown in \hl{Reference a Box here?  or possibly Appendix A.x?}


% In many cases, The American Naturalist allows authors to typeset 
% their own supplementary material in an author-supplied PDF. For author-
% supplied PDFs, please consult the AmNat_supp_template.tex document,
% available from https://www.journals.uchicago.edu/journals/an/instruct 
%
% By contrast, the Appendix instructions below apply to cases in which
% supplementary material is to be typeset by the AmNat editorial staff.
% That notably includes descriptions of methods, tables defining parameters,
% and other material necessary for reproducing the MS's results.
%
% Please reset counters for the appendix (thus normally figure A1, 
% figure A2, table A1, etc.).
%
% In certain cases, it may be appropriate to have a PRINT appendix in
% addition to (or instead of) an online appendix. In this case, please 
% name the print appendix Appendix A, and any subsequent appendixes (if 
% there are any) should be named Online Appendix B, Online Appendix C,
% etc.
%
% Counters for each appendix should match the letter of that appendix.
% For example, tables in Appendix C should be numbered table C1, table C2,
% etc. This applies to tables, equations, and figures.
%
% It's better not to use the \appendix command, because we have some
% formatting peculiarities that \appendix conflicts with.

\renewcommand{\theequation}{A\arabic{equation}}
\renewcommand{\thetable}{A\arabic{table}}
\setcounter{equation}{0}  % reset counter 
\setcounter{figure}{0}
\setcounter{table}{0}

\section{Appendix B: Additional Methods}


%%%%%%%%%%%%%%%%%%%%%
% Bibliography
%%%%%%%%%%%%%%%%%%%%%
% You can either type your references following the examples below, or
% compile your BiBTeX database and paste the contents of your .bbl file
% here. The amnatnat.bst style file should work for this---but please
% let us know if you run into any hitches with it!
%
% If you upload a .bib file with your submission, please upload the .bbl
% file as well; this will be required for typesetting.
%
% The list below includes sample journal articles, book chapters, and
% Dryad references.
\bibliography{Refs2.bib}

\newpage{}



%%%%%%%%%%%%%%%%%%%%%
% Tables
%%%%%%%%%%%%%%%%%%%%%
\section{Tables}
\renewcommand{\thetable}{\arabic{table}}
\setcounter{table}{0}


%%%%%%%%%%%%%%%%%%%%%%%%
\begin{table}[htbp]
\centering
\caption{\bf Definition of terms.}
\label{tab:Terms}
\begin{tabular}{ l l r }
 \toprule
Symbol & Definition & Dimension \\
\hline
$g$      & number of genotypes ($3$; $AA$, $Aa$, and $aa$) & \\
$\omega$ & number of stages ($2$; juvenile and adult) & \\
$\tilde{\mbf{n}}$ & Joint stage $\times$ genotype vector & $2 \omega g \times 1$ \\
$\tilde{\mbf{p}}$ & Joint stage $\times$ genotype frequency vector & $2 \omega g \times 1$ \\

$\mbf{U}^{S}_{i}$, $\mbf{U}^{X}_{i}$ & Genotype-specific transition and survival matrices & $\omega \times \omega$ \\
$\mbf{F}_{i}$, $\mbf{F}^{\prime}_{i}$ & Genotype-specific fertility matrices & $\omega \times \omega$ \\
$\mbf{H}^{S}_{i}$, $\mbf{H}^{X}_{i}$ & Parent-offspring genotype maps & $g \times g$ \\
$\mbf{W}$   & Allele segregation matrix & $\omega \times g$ \\
$\mbf{K}$   & Vec-permutation matrix & $g \times g$ \\
$\mbf{I}$   & Identity matrix & given in subscript \\
\hline
\end{tabular}
\end{table}
\newpage{}




\begin{table}[h]
\caption{Founders of \textit{The~American Naturalist}}
\label{Table:Founders}
\centering
\begin{tabular}{lll}\hline
Early editor            & Years with the journal \\ \hline
Alpheus S. Packard Jr.  & 1867--1886 \\
Frederick W. Putnam     & 1867--1874 \\ 
Edward S. Morse         & 1867--1871 \\ 
Alpheus Hyatt           & 1867--1871 \\
Edward Drinker Cope$^a$ & 1878--1897 \\
J.~S. Kingsley          & 1887--1896 \\ \hline 
\end{tabular}
\bigskip{}
\\
{\footnotesize Note: Table titles should be short. Further details should go in a `notes' area after the tabular environment, like this. $^a$ Published the first description of \textit{Dimetrodon}.}
\end{table}

\newpage{}

\section{Figure legends}

\begin{figure}[h!]
%\includegraphics{horn-of-okapi}
\caption{Figure legends can be longer than the titles of tables. However, they should not be excessively long.}
\label{Fig:OkapiHorn}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%
% Videos
%%%%%%%%%%%%%%%%%%%%%
% If you have videos, journal style for them is similar to that for
% figures. You'll want to include a still image (such as a JPEG)
% to give your readers a preview of what the video looks like.

%%%%% Include the text below if you have videos

\renewcommand{\figurename}{Video} 
\setcounter{figure}{0}
% Thanks to Flo Debarre for the pro tip of putting
% \renewcommand{\figurename}{Video} before the Video legend and
% \renewcommand{\figurename}{Figure} after it!

\begin{figure}[h!]
%\includegraphics{VideoScreengrab.jpg}
\caption{Video legends can follow the same principles as figure legends. Counters should be set and reset so that videos and figures are enumerated separately.}
\label{VideoExample}
\end{figure}

\renewcommand{\figurename}{Figure}
\setcounter{figure}{1}

%%%%% Include the above if you have videos


\begin{figure}[h!]
%\includegraphics{elegance}
\caption{In this way, figure legends can be listed at the end of the document, with references that work, even though the graphic itself should be included for final files after acceptance. Instead, upload the relevant figure files separately to Editorial Manager; Editorial Manager should insert them at the end of the PDF automatically.}
\label{Fig:AnotherFigure}
\end{figure}

\subsection{Online figure legends}

\renewcommand{\thefigure}{A\arabic{figure}}
\setcounter{figure}{0}

\begin{figure}[h!]
%\includegraphics{jumps20m}
\caption{\textit{A}, the quick red fox proceeding to jump 20~m straight into the air over not one, but several lazy dogs. \textit{B}, the quick red fox landing gracefully despite the skepticism of naysayers.}
\label{Fig:Jumps}
\end{figure}

\begin{figure}[h!]
%\includegraphics{jumps20m}
\caption{The quicker the red fox jumps, the likelier it is to land near an okapi. For further details,.}
\label{Fig:JumpsOk}
\end{figure}

\renewcommand{\thefigure}{B\arabic{figure}}
\setcounter{figure}{0}

\end{document}
