\documentclass[11pt]{article}
% Preamble
\usepackage[sc]{mathpazo} %Like Palatino with extensive math support
\usepackage{fullpage}
\usepackage[authoryear,sectionbib,sort]{natbib}
\bibliographystyle{amnatnat.bst}
\setlength{\bibsep}{0.0pt}
\linespread{1.7}
\usepackage[utf8]{inputenc}
\usepackage[left]{lineno}
\usepackage{titlesec}
\usepackage{amsmath}
\usepackage{amsfonts}
\usepackage{amssymb}
\usepackage[utf8]{inputenc}
\usepackage{color,soul}
\usepackage{booktabs}
\usepackage{tikz}
\usepackage{pdflscape}
\usepackage[colorlinks=true, allcolors=black]{hyperref}
\usepackage{nameref}

% Section Header Formats
\titleformat{\section}[block]{\Large\bfseries\filcenter}{\thesection}{1em}{}
\titleformat{\subsection}[block]{\Large\itshape\filcenter}{\thesubsection}{1em}{}
\titleformat{\subsubsection}[block]{\large\itshape}{\thesubsubsection}{1em}{}
\titleformat{\paragraph}[runin]{\itshape}{\theparagraph}{1em}{}[. ]\renewcommand{\refname}{Literature Cited}

% Special Math Characters
\newcommand\encircle[1]{%
  \tikz[baseline=(X.base)] 
    \node (X) [draw, shape=circle, inner sep=0] {\strut #1};}


% Graphics package
\usepackage{graphicx}
\graphicspath{{../output/figs/}.pdf}

% Change default margins
\usepackage[top=0.75in, bottom=0.75in, left=0.75in, right=0.75in]{geometry}

% Definitions
\def\mathbi#1{\textbf{\em #1}}
\def\mbf#1{\mathbf{#1}}
\def\mbb#1{\mathbb{#1}}
\def\mcal#1{\mathcal{#1}}
\newcommand{\bo}[1]{{\bf #1}}
\newcommand{\tr}{{\mbox{\tiny \sf T}}}
\newcommand{\bm}[1]{\mbox{\boldmath $#1$}}

 \def\linenumberfont{\normalfont\scriptsize}
%===========================================
% Basic info from Journal

%%%%%%%%%%%%%%%%
% Line numbering
%%%%%%%%%%%%%%%%

% Please use line numbering with your initial submission and
% subsequent revisions. After acceptance, please turn line numbering
% off by adding percent signs to the lines %\usepackage{lineno} and
% to %\linenumbers{} and %\modulolinenumbers[3] below.
%
% To avoid line numbering being thrown off around math environments,
% the math environments have to be wrapped using
% \begin{linenomath*} and \end{linenomath*}
%
% (Thanks to Vlastimil Krivan for pointing this out to us!)

%%%%%%%%%%%%
% Authorship
%%%%%%%%%%%%
% Please remove authorship information while your paper is under review,
% unless you wish to waive your anonymity under double-blind review. You
% will need to add this information back in to your final files after
% acceptance.

% The journal does not have numbered sections in the main portion of
% articles. Please refrain from using section references (à la
% section~\ref{section:CountingOwlEggs}), and refer to sections by name
% (e.g. section ``Counting Owl Eggs'').

% You may wish to remove the Acknowledgments section while your paper 
% is under review (unless you wish to waive your anonymity under
% double-blind review) if the Acknowledgments reveal your identity.
% If you remove this section, you will need to add it back in to your
% final files after acceptance.

%If you have deposited data to Dryad, you should cite them somewhere in the main text (usually in the Methods or Results sections). A sentence like the following will do. All data are available in the Dryad Digital Repository (\citealt{CookEtAl2015}).

%===========================================

% This version of the LaTeX template was last updated on
% November 8, 2019.

\begin{document}
\title{Evolutionary demography and the maintenance of sexually antagonistic polymorphism in outcrossers and simultaneous hermaphrodites}
\author{Colin Olito$^{1,\ast}$ \\ 
Charlotte de Vries$^{2}$}
\date{\today}
\maketitle

\noindent{} 1. Department of Biology, Lund University, Lund 223 62, Sweden;

\noindent{} 2.  Department of Evolutionary Biology and Environmental Studies, University of Zurich, Zurich CH-8057, Switzerland;

\noindent{} $\ast$ Corresponding author; e-mail: colin.olito@gmail.com

\bigskip

\textit{Manuscript elements}: Figure~1, figure~2, table~1, online appendices~A and B (including figure~A1 and figure~A2). Figure~2 is to print in color.

\bigskip

\textit{Keywords}: Intralocus sexual conflict, Evolutionary Demography, Balancing selection, Hermaphrodite, mixed mating systems, inbreeding depression 

\bigskip

\textit{Manuscript type}: Article. %Or e-article, note, e-note, natural history miscellany, e-natural history miscellany, comment, reply, invited symposium, or historical perspective.

\bigskip

\noindent{\footnotesize Prepared using the suggested \LaTeX{} template for \textit{Am.\ Nat.}}

\linenumbers{}
\modulolinenumbers[3]

\newpage{}


%====================
% Begin Main Text
%====================

%\section*{Outline}

%OK, just putting some ideas out here based roughly on our conversations about an {\itshape AmNat}-style article, the results we have so far, and what I think we should be able to do with data from COMPADRE. 

%\begin{enumerate}
	%\item {\bf Introduction}
	%\begin{enumerate}
%		\item Briefly introduce sexual antagonism and intra-locus sexual conflict. (Colin)
%		\item Simultaneous hermaphrodites, mixed mating systems, unique predictions for SA polymorphism. (Colin)
%		\item Introduce Evolutionary Demography, pivot to focus on the consequences of taking demography into account explicitly for the maintenance of SA polymorphism. (Lotte)
%		\item Introduce key difference between existing population genetic models of SA polymorphism and our models: possibility of diverse population and evolutionary dynamics. (Lotte then Colin)
%		\item Explain and highlight "demographically viable" parameter space, indicate that we are interested in whether parameter space predicted to be polymorphic by Pop.~gen.~models is, indeed, "demographically viable" (Lotte then Colin)
%		\begin{enumerate}
%			\item Pop. Gen. models tacitly assume stable population size, parameterize genotypic fitnesses relative to the 'best performing' genotype. Non-overlapping generations. Casually define fitness as lifetime reproductive fitness of a given genotype.
%			\item "In practice, these genotypic fitnesses emerge from myriad process acting throughout the life-history of individuals"
%			\item these processes contribute to key demographic rates such as the intrinsic population growth rate, 
%			\item 
%		\end{enumerate}
%		\item Building upon the two-sex stage-structured matrix models developed by \citet{deVriesCaswell2019a,deVriesCaswell2019b}, we develop stage-structured matrix models that can accommodate selection through male and female function in simultaneous hermaphrodites with mixed mating systems. The models also allow for the effects of inbreeding depression to manifest in several demographic parameters, allowing us to explore the fitness consequences of early- vs.~late-acting inbreeding depression. (Colin then Lotte)
	%\end{enumerate}
	%\item {\bf Methods}
%		\begin{enumerate}
%			\item Citation heavy review of de Vries \& Caswell AmNat (2018) \& de Vries \& Caswell AmNat (2019) TheorPopBiol
%			\item Briefly describe extension to simultaneous hermaphrodites, citing Jordan \& Connallon (2014), Abbot \& Tazzyman (2014), Olito (2017). 
%			\item Breakdown into individuals produced by selfing vs.~outcrossing.
%			\item Brief anatomy of $\mathbf{\tilde{A}}$.
%			\item Analysis. Invasion analysis \& abridged derivation of invasion conditions \& identification of extinction threholds.
%			\item Cite Appendix with full derivation distilled from notes file.
%		\end{enumerate}
	%\item {\bf When is polymorphism demographically viable?}
	%\begin{enumerate}
%		\item Review previous predictions for outcrossers \& partial selfing
%		\begin{enumerate}
%			\item Greater scope for SA polymorphism in outcrossers vs.~selfers
%			\item Greater scope for SA polymorphism when SA fitness effects are recessive (dominance reversal).
%			\item but our model allows us to see where/when invasion of SA alleles can drive the population extinct
%			\item \textbf{Fig.1 - illustration of funnel plots w/ extinction isoclines}
%		\end{enumerate}
%		\item Key results summarized in \textbf{Fig.~2}: 
%		\begin{enumerate}
%			\item For simplicity, temporarily assume no inbreeding depression ($\delta = 0$).
%			\item When populations do not have high growth-rates (i.e., when $\lambda \approx 1$), much of of polymorphic parameter space becomes demographically inviable.
%			\item Specifically, would-be polymorphic paramter space becomes inviable when segregating male-beneficial/female-deleterious alleles can reduce fecundity enough to drive the population extinct.
%			\item This effect is most dramatic for separate-sexed species and predominant outcrossers. Selfing shelters the population from the female-deleterious fitness effects because there is no opportunity for selection via male function. This can also be thought of as reproductive assurance in our model....
%			\item Convergence to pop.~gen.~predictions when $f$ is big.
%		\end{enumerate}
	%\end{enumerate}
	%\item {\bf Effects of inbreeding depression}
	%\begin{enumerate}
%		\item Review effect of I.D.~in population genetic models: causes selfing populations to look more like outcrossing ones (at least in terms of where SA polymorphism is maintained). This is because a fraction of selfed ovules are aborted, meaning the overall ratio of outcrossed vs.~selfed progeny increases.
%		\item Matrix model allows us to explore effects of I.D. in different parts of the life-cycle (early- vs.~late-acting I.D.). Point out that previous pop.~gen.~models only address early-acting I.D. via ovule viability.
%		\item MAKE CLEAR that purging of deleterious recessives thought to cause I.D. is already included in the model results presented in Fig.~3
%		\item Key results summarized in Fig.~3
%		\begin{enumerate}
%			\item Pop.~Gen.~models predict that higher early-acting ID should lead to greater proportion of polymorphic parameter space, while our models predict that we get a critical threshold of selfing at which population viability plummets.
%			\item When and how I.D. influences fitness during the life-cyclecan change the location and steepness of the decline in polymorphic parameter space.
%			\item Critically, largest deviations from pop.~gen.~predictions happen for predominantly selfing populations.
%			\item A perhaps obvious, but sometimes overlooked implication of our model predictions: selfing can offer reproductive assurance, and SA polymorphisms can be maintained in a non-trivial proportion of parameter space in partially selfing populations... BUT demographically, the population must be able to "afford" the loss of selfed ovules/offspring.
%			\item HIGHLIGHT that this result holds even though we incorporated decrease of the mutation load due to inbreeding (and corresponding decrease in I.D.)...	
%		\end{enumerate}
	%\end{enumerate}
	%\item {\bf Mimulus as a potential real world example}
	%\begin{enumerate}
%		\item We can parameterize our model with real populations' demographic rates, and make explicit predictions regarding the opportunity for SA polymorphism given the species' demography. We provide two illustrative examples:
%		\item Opportunities for SA polymorphism in {\itshape Mimulus guttatus} (now {\itshape Erythranthe guttata})
%		\item Possibly use data from \citet{PetersonEtAl2016} Eagle Meadows population, which is the 'local' population in their big common garden local adaptation expeirment.
%		\begin{enumerate}
%			\item Illustrate funnel plot using {\itshape E.~gutatta} demographic rates.
%			\item \textbf{Fig.~4}: PLOT inversion from \citet{LeeKelly2015}, on that funnel!
%		\end{enumerate}
	%\end{enumerate}
	%\item Discussion
%		\begin{enumerate}
%			\item Classic prediction from demography that most species should have a $\lambda \approx 1$
%			\item If true, the demographic impacts on the opportunities for the maintenance of sexually antagonistic polymorphisms may be pervasive...
%			\item Condition dependence, fecundity, scope for sexual confict/antagonistic polymorphism?
%			\item Do sexually dimorphic species exhibit demographic rates that increase opportunities for SA polymorphism?
%		\end{enumerate}
%\end{enumerate}





%%%%%%%%%%%%%%%%%%%%
\newpage{}
\section*{Abstract}
%%%%%%%%%%%%%%%%%%%%
\ldots 

\newpage{}

%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Introduction}
%%%%%%%%%%%%%%%%%%%%%%%%

Whether a population persists or goes extinct depends in large part on its ability to adapt to the environment. Yet, the process of adaptation can be impeded by a variety of genetic and environmental factors, including deleterious mutations \citep{Haldane1957}, environmental shifts causing a population to chase a new phenotypic optimum \citep{Maynard-Smith1976, LandeShannon1996,OrrUnckless2008}, maladaptive gene-flow \citep{KirkpatrickBarton1997, BolnickNosil2007}, as well as genetic constraints arising from intrinsic features of the inheritance system \citep{ConnallonHall2018}. 

Conflicting selection and gene-flow between different classes of individuals, or genetic trade-offs between fitness components, can impose constraints on adaptation with particularly interesting evolutionary consequences \citep{CharlesworthHughes2000, ConnallonHall2018}. Such constraints hinder adaptation by preventing individuals (or classes of individuals) from reaching the phenotypic optimum in one or more fitness components, thereby increasing the population's overall extinction risk. At the same time, they provide an effective mechanism for the maintenance of genetic variation, which in turn influences the population's capactiy for future adaptation \citep{Fisher1930, CharlesworthHughes2000, ConnallonHall2018, MatthewsConnallon2019}. Hence, for genetically constrained traits the nature and extent of genetic variation observed in natural populations should reflect the balance between the contribution of constraints to the maintainance of genetic polymorphisms, and the demographic consequences of the resulting maladaptation.
%Genetic trade-offs between fitness components due to pleiotropy represent a form of genetic constraint with particularly interesting evolutionary consequences because they act as a double-edged sword
%The nature and extent of genetic variation observed in natural populations should therefore reflect the balance between the contribution of those constraints to the maintainance of genetic polymorphism and the demographic consequences of the resulting maladaptation.

Genetic trade-offs between male and female fitness are a common feature of sexually reproducing populations, and are thought to contribute not only to the maintenance of genetic variation, but also the evolution of sexual dimorphism and gender-related traits \citep{Lande1980, Rice1992, Charlesworth1999, RiceChippindale2001, BondurianskyChenoweth2009,Olito2019}. When such genetic trade-offs occur, "sexually antagonistic" selection (abbreviated SA hereafter) can arise when beneficial alleles for one sex are deleterious when expressed in the other \citep{Kidwell1977, Rice1992, ConnallonClark2012}. Moreover, alleles with opposing fitness effects through male and female sex functions create analogous genetic constraints on fitness in hermaphrodite populations, where both maternal and paternal reproductive success contribute jointly to each individuals' overall fitness \citep{LloydWebb1986, WebbLloyd1986, Abbott2011, JordanConnallon2014}. 

The evolutionary dynamics of SA alleles in hermaphrodites differ from those in dioecious populations because individuals may reproduce through a combination of self- and outcross fertilization \citep{Goodwillie2005, Igic2006, JarneAuld2006}. Such mixed-mating systems have important theoretical consequences for the maintenance of SA polymorphisms in hermaphrodite populations. Self-fertilization is predicted to reduce the total parameter space where balanced SA polymorphisms can be maintained, while simultaneously creating a bias in selection through the female sex function (\citealt{JordanConnallon2014}; but see \citealt{Tazzyman2015}. Yet, other processes, such as genetic linkage to other SA loci or a sex-determining region \citep{Otto2011, JordanCharlesworth2012, Olito2017, Olito2019}, or spatial heterogeneity and the complexity of the life-cycle \citep{Olito-etal-2018,ConnallonSharmaOlito2019} can expand the parameter space for SA polymorphism. Overall, both theoretical predictions and current empirical data suggest there is ample scope for SA trade-offs and the maintenance of SA polymorphisms in both dioecious and hermaphrodite populations \citep{Abbott2011, WangBarrett2020}, although identifying specific SA loci from genome sequence data remains challenging \citep{RuzickaESEB2020}.
\bigskip

\begin{enumerate}
	\item Introduce Evolutionary Demography, pivot to focus on the consequences of taking demography into account explicitly for the maintenance of SA polymorphism. [Lotte]
	\item Introduce key difference between existing population genetic models of SA polymorphism and our models: possibility of diverse population and evolutionary dynamics. [Lotte then Colin]
	\item Explain and highlight "demographically viable" parameter space, indicate that we are interested in whether parameter space predicted to be polymorphic by Pop.~gen.~models is, indeed, "demographically viable" [Lotte then Colin]
		\begin{enumerate}
			\item Pop. Gen. models tacitly assume large and/or stable population size, parameterize genotypic fitnesses relative to the 'best performing' genotype. Non-overlapping generations. Casually define fitness as lifetime reproductive fitness of a given genotype.
			\item "In practice, these genotypic fitnesses emerge from myriad process acting throughout the life-history of individuals"
			\item these processes contribute to key demographic rates such as the intrinsic population growth rate, 
		\end{enumerate}
		\item A major strength of the model is that we can use empirically estimated demographic rates to parameterize the model, enabling us to make predictions about the scope for demographically viable SA polymorphisms that are grounded in the biology of real populations.
\end{enumerate}
\bigskip

In this article, we build upon the models of \citet{deVriesCaswell2019a} and \citet{deVriesCaswell2019b} to develop a stage-structured mendelian matrix model that accommodates selection through male and female function in simultaneous hermaphrodites with mixed mating systems. Using the model, we jointly analyze the evolutionary and demographic consequences of SA selection in obligately outcrossing (equivalent to dioecious) and partially selfing hermaphrodite populations. Although a variety of interesting eco-evolutionary dynamics are possible in the model (e.g., \citealt{deVriesCaswell2019a, deVriesCaswell2019b,CaswellEtAl2018}), we focus our analyses on identifying the parameter conditions under which SA polymorphisms are maintained by balancing selection {\itshape and} the population is also demographically viable (i.e., has a positive intrinsic growth rate). The model also allows for the effects of inbreeding depression to manifest in different life-history stages, enabling us to explore the fitness consequences of early- vs.~late-acting inbreeding depression on the model predictions. Finally, we give an illustrative example of the potential for SA polymorphism in real populations by parameterizing our model with empirically estimated demographic rates for the hermaphroditic flowering plant {\itshape Mimulus guttatus} (now {\itshape Erythranthe guttata}).

A key finding from our analysis is that when the intrinsic population growth rate is close to $1$, the deleterious effects of segregating male-beneficial SA alleles on female fecundity can result in extinction over much of the parameter space for SA polymorphism. Furthermore, this demographically viable polymorphic parameter space is often biased towards 'female-beneficial/male-deleterious' SA alleles (i.e., those with weaker selection on the female- than male sex function). Under some conditions, self-fertilization can alleviate the demographic costs of balanced SA polymorphisms, but the concommitant effects of inbreeding depression greatly exacerbate them. Interestingly, our illustrative example using {\itshape M.~guttatus} appears to show that demographic rates observed in real populations can sustain large regions of viable SA polymorphic space. Overall, our findings provide a more nuanced picture of the nature of SA genetic variation that we should expect to find in natural populations, where the fate of SA alleles and the populations harboring them is determined jointly by evolutionary and demographic processes.


%%%%%%%%%%%%%%%%%%%%%%%%
\section*{The Model}
%%%%%%%%%%%%%%%%%%%%%%%%

Here, we briefly derive a matrix model incorporating multiple life-cycle stages and a single diallelic locus under SA selection on male and female sex functions for a population of partially selfing simultaneous hermaphrodites. The derivation and some of the key results follow closely those presented in \citet{deVriesCaswell2019a} and \citet{deVriesCaswell2019b}; and in fact, the model presented here reduces to the two-sex stage-structured model of \citet{deVriesCaswell2019b} under obligate outcrossing. A full derivation of the model and analyses is presented in \hl{Appendix A}, and all simulation code necessary to reproduce the results are available at \url{https://github.com/colin-olito/SA-Hermaphrodites-wDemography}.

In the model, individuals are classified jointly by life-cycle stage ($1, \ldots, \omega$), genotype ($1, \ldots, g$), and whether they were produced by self- or outcross fertilization (denoted by $S$ and $X$ superscripts). Each genotype is characterized by a matrix of transition probabilities (including survival) and a matrix of reproductive output through male and female function. These matrices can include time variation or nonlinearities reflecting the environment or density dependence, but these will not be modelled here. Each stage contributes offspring to genotypes at the next time step according to transition matrices that are determined by the mating system and stage-structure of the population.

The population state at time $t$ is described by a stage $\times$ genotype distribution matrix, which can be transformed into a population state vector, $\tilde{\mbf{n}}(t)$, which is ordered by genotype, then stage, then by how individuals were produced (self vs.~outcrossing). For a single diallelic locus with alleles $A$ and $a$, we have three genotypes ($AA,\, Aa,\, aa$; $g = 3$), giving the population state vector:
\begin{linenomath*}
\begin{equation} \label{eq:PopStateVec}
	\tilde{\mbf{n}}(t) =  \left[
								\begin{array}{c}
									\mbf{n}^{S}_{AA}(t) \\
									\mbf{n}^{S}_{Aa}(t) \\
									\mbf{n}^{S}_{aa}(t) \\ \hline
									\mbf{n}^{X}_{AA}(t) \\
									\mbf{n}^{X}_{Aa}(t) \\
									\mbf{n}^{X}_{aa}(t) \\ 
						\end{array} \right],
\end{equation}
\end{linenomath*}

\noindent where $\mbf{n}^{S}_{i}$ and $\mbf{n}^{X}_{i}$ ($i \in \{AA,Aa,aa\}$) are the stage $\times$ genotype distribution vectors of individuals produced by self-fertilization and outcrossing, respectively. The proportional population vector is given by
\begin{linenomath*}
\begin{equation} \label{eq:propPopVec}
	\tilde{\mbf{p}}(t) = \frac{\tilde{\mbf{n}}(t)}{ \| \tilde{\mbf{n}}(t) \|} = \mbf{p}(t),
\end{equation}
\end{linenomath*}

\noindent where $\| \cdot \|$ is the one-norm. The population vector $\tilde{\mbf{n}}(t)$ is projected forward from time $t$ to $t + 1$ by the $projection$ matrix $\tilde{\mbf{A}}(\tilde{\mbf{n}})$ such that 
\begin{linenomath*}
\begin{equation}
	\tilde{\mbf{n}}(t + 1) = \tilde{\mbf{A}}[\tilde{\mbf{n}}(t)] \, \tilde{\mbf{n}}(t)
\end{equation}
\end{linenomath*}

The population projection matrix $\tilde{\mbf{A}}$ is constructed from four sets of matrices representing the demographic and genetic processes: The matrices $\mbf{U}^{S}_{i}$ and $\mbf{U}^{X}_{i}$ contains transition and survival probabilities for each genotype, produced by selfing and outcrossing respectively. The matrices $\mbf{F}_{i}$ and $\mbf{F}^{\prime}_{i}$ contain the genotype $\times$ stage specific contribution via female and male sex-functions of genotype $i$ to the female and male gamete pools, respectively, and therefore to zygotes in the next generation. In principle, the model can admit any kind of pleiotropy among demographic traits because the transition and fertility matrices can differ among genotypes in any way. For simplicity, we assume linear, time-invariant demography.

Similar to the two-sex model of \citet{deVriesCaswell2019b}, we define the matrices $\mbf{H}^S_{j}(\tilde{\mbf{n}})$ and $\mbf{H}^X_{j}(\tilde{\mbf{n}})$, which map the genotypes of the parent in stage $j$ to the genotypes of their offspring produced by selfing and outcrossing, respectively. The $(k, l)$ entry of $\mbf{H}^{X}_{j}$ and $\mbf{H}^{S}_{i}$ is the probability that an offspring of a genotype $l$ mother, of stage $i$, has genotype $k$. For the purpose of this article, we assume that mating is random with respect to stage and hence that the parent-offspring map is the same for all stages (i.e., $\mbf{H}^S_{j}(\tilde{\mbf{n}}) = \mbf{H}^S(\tilde{\mbf{n}})$, and $\mbf{H}^X_{j}(\tilde{\mbf{n}}) = \mbf{H}^X(\tilde{\mbf{n}})$). The matrices $\mbf{H}^S(\tilde{\mbf{n}})$ and $\mbf{H}^X(\tilde{\mbf{n}})$ contain the population genetic processes and are presented in the next section.

%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Mating and offspring production under partial selfing}

Simultaneous hermaphrodites express both male and female sex functions (i.e., can be both a mother and father), and have the potential to reproduce by a combination of self- and outcross-fertilizations. Maternal outcrossing involves receiving male gametes from another individual in the population, while paternal outcrossing involves exporting male gametes to another individual and fertilizing their ovules. Self-fertilization is acheived when an individual's male gametes fertilize their own ovules. These modes of reproduction complicate mating in our model relative to the two-sex stage-structured model of \citet{deVriesCaswell2019b}.

Let $C$ denote the proportion of each individual's ovules that are self-fertilized (the remaining $1 - C$ are outcrossed), and $\delta$ represents the proportion of self-fertilized zygotes that fail to develop due to inbreeding depression during early development \citep{Charlesworth1987}. In \hl{Appendix A.x}, we present a more general model of genotype-specific self-fertilization rates (after \citealt{JordanConnallon2014}). Outcrossing proceeds similarly to a two-sex model of reproduction \citep{deVriesCaswell2019b}, except each individual's genotype determines both the number of ovules produced, and also paternal "mating success", broadly defined. For hermaphroditic flowering plants, for example, paternal mating success could reflect pollen production, export efficiency, pollen-tube germination and growth rates, among other things \citep{LloydWebb1986,WangBarrett2020,Harder2016}. Depending on their stage and genotype at the SA locus, individuals contribute male gametes to the overall population gamete pool according to the male stage $\times$ genotype vector:
\begin{linenomath*}
\begin{equation}
		\mbf{W}^{\prime} \mbb{F}^{\prime} \mbf{p} = 
		\left(
			\begin{array}{ccc}
				\mbf{1}^{\intercal}_{\omega} & \frac{1}{2} \mbf{1}^{\intercal}_{\omega} & 0 \\
				0 & \frac{1}{2} \mbf{1}^{\intercal}_{\omega} & \mbf{1}^{\intercal}_{\omega} \\
			\end{array} \right)
		\left(
			\begin{array}{ccc}
				\mbf{F}^{\prime}_{AA} & 0 & 0 \\
				0 & \mbf{F}^{\prime}_{Aa} & 0 \\
				0 & 0 & \mbf{F}^{\prime}_{aa} \\
			\end{array} \right)
		\left(
			\begin{array}{c}
				\mbf{p}_{AA} \\
				\mbf{p}_{Aa} \\
				\mbf{p}_{aa} \\
			\end{array} \right).
\end{equation}
\end{linenomath*}

\noindent The vector $\mbf{p}$ gives the genotype frequencies at mating, and is the sum of $\mbf{n}^{S}$ and $\mbf{n}^{X}$, since in partially selfing popualtions, whether an individual was produced by selfing or outcrossing does not affect whether they can reproduce by outcrossing themselves. The matrix $\mbb{F}^{\prime}$ is composed of genotype-specific fertility matrices, and operates on the the vector of genotype frequencies to give their relative contribution of each genotype to the gamete pool. The matrix $\mbf{W}^{\prime}$ converts these contributions to allele numbers. Normalizing the resulting vector gives the allele frequencies in the male gamete pool:
\begin{linenomath*}
\begin{equation} \label{eq:maleGametePool}
	\left(
		\begin{array}{c}
			q^{\prime}_{A} \\
			q^{\prime}_{a} \\
		\end{array} \right) = 
			\frac{\mbf{W}^{\prime} \mbb{F}^{\prime} \mbf{p}}{\| \mbf{W}^{\prime} \mbb{F}^{\prime} \mbf{p} \|} = 
				\frac{\mbf{W}^{\prime} \mbb{F}^{\prime} (\mbf{n}^X + \mbf{n}^S)}{\| \mbf{W}^{\prime} \mbb{F}^{\prime} (\mbf{n}^X + \mbf{n}^S)\|}
\end{equation}
\end{linenomath*}

The parent-offspring map for outcrossing is a function of the allele frequencies in the male gamete pool, and is given by 
\begin{linenomath*}
\begin{equation} \label{eq:HX}
	\mbf{H}^X(\tilde{\mbf{n}}) = 
			\left(
			\begin{array}{ccc}
				q^{\prime}_{A} & \frac{1}{2} q^{\prime}_{A} & 0 \\
				q^{\prime}_{a} & \frac{1}{2} & q^{\prime}_{A}  \\
				0 & \frac{1}{2} q^{\prime}_{a} & q^{\prime}_{a} \\
			\end{array} \right)
\end{equation}
\end{linenomath*}

\noindent From left to right, the columns of matrix $\mbf{H}^X(\tilde{\mbf{n}})$ give the genotype distribution of outcrossed offspring produced by a maternal parent of each genotype ($AA$, $Aa$, and $aa$ respectively).

Reproduction via self-fertilization differs from outcrossing. As in previous models of partial selfing, we assume that individuals produce enough male gametes to easily self-fertilize all of their ovules, and that self-fertilization involves little or no selection from external factors relative to outcrossing (e.g., \citealt{Charlesworth1978a,JordanConnallon2014,Olito2017}, but see \citealt{Tazzyman2015}). Under these assumptions, the genotype distributions of selfed offspring are determined entirely by the parental genotype and the probabilities of segregation and fertilization during meiosis: 
\begin{linenomath*}
\begin{equation} \label{eq:HS}
	\mbf{H}^S(\tilde{\mbf{n}}) = 
			\left(
			\begin{array}{ccc}
				1 & 1/4 & 0 \\
				0 & 1/2 & 0 \\
				0 & 1/4 & 1 \\
			\end{array} \right).
\end{equation}
\end{linenomath*}

\noindent The columns of $\mbf{H}^S(\tilde{\mbf{n}})$ again give the selfed offspring genotype distributions for parental genotypes of $AA$, $Aa$, and $aa$, respectively.


%%%%%%%%%%%%%%%%%%%%%%
\subsubsection*{Population projection}

Using the component matrices described in the previous section, the matrix $\tilde{\mbf{A}}[\tilde{\mbf{n}}]$ that projects the eco-evolutionary dynamics is:
\begin{linenomath*}
\begin{equation} \label{eq:Atilde}
	\tilde{\mbf{A}}[\tilde{\mbf{n}}] = 
			\underbrace{\left(
			\begin{array}{c|c}
				\mcal{U}^S & \mbf{0} \\ \hline
				\mbf{0} & \mcal{U}^X \\
			\end{array} \right)}_{\tilde{\mbf{U}}} + 
			\underbrace{\left(
			\begin{array}{c|c}
				\mcal{F}^S(\tilde{\mbf{p}}) C(1 - \delta) & \mcal{F}^S(\tilde{\mbf{p}}) C(1 - \delta) \\ \hline
				\mcal{F}^X(\tilde{\mbf{p}}) (1 - C) & \mcal{F}^X(\tilde{\mbf{p}}) (1 - C)\\
			\end{array} \right)}_{\tilde{\mbf{F}}}.
\end{equation}
\end{linenomath*}

\noindent The blocks of the component matrices in Eq(\ref{eq:Atilde}) correspond to production of offspring by self-fertilization and outcrossing ($\mcal{F}^S$ and $\mcal{F}^X$ in $\tilde{\mbf{F}}$), and survival of selfed and outcrossed offspring ($\mcal{U}^S$ and $\mcal{U}^X$ in $\tilde{\mbf{U}}$). 

We construct $\tilde{\mbf{A}}[\tilde{\mbf{n}}]$ using the vec-permutation approach of \citet{CaswellEtAl2018}. Briefly, we construct a set of block-diagonal matrices: e.g.,
\begin{linenomath*}
\begin{equation} \label{eq:BlkUS}
	\mbb{U}^S = 
		\left(
			\begin{array}{ccc}
				\mbf{U}^{S}_{AA} & \mbf{0} & \mbf{0} \\
				\mbf{0} & \mbf{U}^{S}_{Aa} & \mbf{0} \\
				\mbf{0} & \mbf{0} & \mbf{U}^{S}_{aa} \\
			\end{array} \right),
\end{equation}
\end{linenomath*}

\noindent and corresponding matrices $\mbb{U}^X$, $\mbb{F}^S$, $\mbb{F}^X$. Since individuals cannot change their genotype once they are born, the survival matrices are block diagonal, and $\mcal{U}^S = \mbb{U}^S$ and $\mcal{U}^X = \mbb{U}^X$. 

Similarly, we also construct block-diagonal matrices
\begin{linenomath*}
\begin{equation}
	\mbb{H}^j = \mbf{I}_{\omega} \otimes \mbf{H}^j(\tilde{\mbf{p}}),
\end{equation}
\end{linenomath*}

\noindent where $j \in \{S,X\}$. The corresponding fertility matrices for self-fertilization and outcrossing are defined as
\begin{linenomath*}
\begin{equation} \label{eq:BlkFS}
	\mcal{F}^S = \mbf{K}^{\intercal} \mbb{H}^S(\tilde{\mbf{p}}) \mbf{K} \mbb{F}^S = 
			\left(
			\begin{array}{ccc}
				\mbf{F}_{AA} & \frac{1}{4} \mbf{F}_{Aa} & 0 \\
				0 & \frac{1}{2} \mbf{F}_{Aa} & 0 \\
				0 & \frac{1}{4} \mbf{F}_{Aa} & \mbf{F}_{aa}\\
			\end{array} \right), 
\end{equation}
\end{linenomath*}
\noindent and 
\begin{linenomath*}
\begin{equation} \label{eq:BlkFX}
	\mcal{F}^X = \mbf{K}^{\intercal} \mbb{H}^X(\tilde{\mbf{p}}) \mbf{K} \mbb{F}^X = 
			\left(
			\begin{array}{ccc}
				q^{\prime}_{A} \mbf{F}_{AA} & \frac{1}{2} q^{\prime}_{A} \mbf{F}_{Aa} & 0 \\ 
				q^{\prime}_{a} \mbf{F}_{AA} & \frac{1}{2} \mbf{F}_{Aa} & q^{\prime}_{A} \mbf{F}_{aa}  \\ 
				0 & \frac{1}{2} q^{\prime}_{a} \mbf{F}_{Aa} & q^{\prime}_{a} \mbf{F}_{aa}  \\
			\end{array} \right),
\end{equation}
\end{linenomath*}

\noindent where $q^{\prime}_A$ and $q^{\prime}_a$ are given by Eq(\ref{eq:maleGametePool}), and $\mbf{K}$ is the vec-permutation matrix \citep{HendersonSearle1981}. Further details of the derivations of $\mcal{F}^S$ and $\mcal{F}^X$ are provided in \hl{Appendix A.x}. 

The blocks of $\mcal{F}^X(\tilde{\mbf{p}})$ can be constructed and interpreted as follows: The first row block of the first column produces $AA$ offspring by outcrossing from $AA$ maternal parents. This happens when the $AA$ maternal parent receives a random gamete from the male gamete pool, which happens with probability $q^{\prime}_{A}$. The other blocks can be interpreted similarly.

Combining all the component matrices yields the overall eco-evolutionary projection matrix shown in \hl{Reference a Box here?  or possibly Appendix A.x?}


%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Sexually antagonistic selection \& inbreeding depression} \label{sec:SAsel}

We now construct and analyze a genotype $\times$stage-classified model for a hypothetical species with intralocus sexual conflict via the two sex functions. To keep the model analytically tractable, we assume our hypothetical species has two life stages: juveniles and adults (i.e., $\omega = 2$), and that only adults are reproductively active. Suppose that there is a genetic trade-off between the sex-functions at a single bi-allelic locus such that allele $A$ is beneficial for female fertility but detrimental for male reproductive success (e.g., pollen production), and that allele $a$ has the reverse effect. Following convention, we parameterize the relative fitnesses of each genotype through each sex function ($w_{i}$ and $w^{\prime}_{i}$) to be bounded by $(0,1)$, with dominance and selection coefficients $h_f, s_f$ and $h_m, s_m$ determining the decrease in fertility through each sex function relative to the most fit genotype ($AA$ has highest female fertility, $aa$ the highest male fertility; see Table \ref{tab:Fitness}). The SA locus does not affect survival and transition rates. However, the survival matrices can be used to model the fitness effects of inbreeding depression at later stages of development by allowing different stage-specific survival and transition rates for individuals produced by self-fertilization vs.~outcrossing. By contrast, the parameter $\sigma$ only affects inbreeding depression through viability of selfed ovlues. With this in mind, we define survival matrices for individuals produced by selfing and outcrossing as follows:
\begin{linenomath*}
\begin{equation} \label{eq:US}
	\mbf{U}^S = \left(
					\begin{array}{cc}
						\sigma^S_J (1 - \delta_J) \big(1 - \gamma^S (1 - \delta_{\gamma}) \big) & 0 \\
						\sigma^S_J (1 - \delta_J) \gamma (1 - \delta_{\gamma})      & \sigma^S_A (1 - \delta_S)
					\end{array}
				\right) \\
\end{equation}
\end{linenomath*}
\noindent and 
\begin{linenomath*}
\begin{equation}\label{eq:UX}
	\mbf{U}^X = \left(
					\begin{array}{cc}
						\sigma^X_J(1 - \gamma^X) & 0 \\
						\sigma^X_J \gamma      & \sigma^X_A
					\end{array}
				\right)
\end{equation}
\end{linenomath*}

\noindent where $\sigma_J$ and $\sigma_A$ are the juvenile and adult stage survival rates, $\gamma$ is the maturation rate from juvenile to adult stages, and the corresponding $\delta_S$, $\delta_A$, and $\delta_{\gamma}$ terms denote the proportional decreases in stage-specific survival and transition rates due to inbreeding depression (i.e., deleterious effects of inbreeding at later life-history stages; e.g., \citealt{HarderRoutely2006}). Any or all of these parameters may differ among genotypes, so that selection can operate on stage-specific viability, development, and/or fertility. For simplicity, we assume these are constant among genotypes. 

The fertility matrices through female and male function are
\begin{linenomath*}
\begin{equation}\label{eq:FS}
	\mbf{F}_{i} = \left(
					\begin{array}{cc}
						0 & f w_{i} \\
						0 & 0
					\end{array}
				\right),
\end{equation}
\end{linenomath*}
\noindent and
\begin{linenomath*}
\begin{equation}\label{eq:FX}
	\mbf{F}^{\prime}_{i} = \left(
					\begin{array}{cc}
						0 & f^{\prime} w^{\prime}_{i} \\
						0 & 0
					\end{array}
				\right),
\end{equation}
\end{linenomath*}

\noindent where $f$ and $f^{\prime}$ represent adult fertilities, and $w_{i}$ and $w^{\prime}_{i}$ the genotypic relative fitness expressions for female and male sex-functions (see Table \ref{tab:Fitness}). 

\begin{table}[htbp]
 \centering
 \caption{\bf Fitness expressions for Sexually Antagonistic selection in hermaphrodites ()}
\label{tab:Fitness}
\begin{tabular}{lccc}
 \toprule
					&  \multicolumn{3}{c}{{\textit{Genotype}}} \\ 
\cline{2-4}
					& $AA$			& $Aa$ 					& $aa$ 		\\ \hline
Female function ($w_{i}$):	& $1$		& $1 - h_f s_f$	& $1 - s_f$ \\	
Male function ($w^{\prime}_{i}$):		& $1 - s_m$& $1 - h_m s_m$	& $1$ 		\\	
\hline
\end{tabular}
\end{table}

Iterating the projection matrix, Eq(\ref{eq:Atilde}) with the above demographic matrices, given an initial population state vector, Eq(\ref{eq:PopStateVec}), allows numerical simulation of the eco-evolutionary dynamics for selection operating on any of the sex-function, or stage-specific demographic parameters. As we outline below, we use numerical techniques together with mathematical analyses to study the conditions for the maintenance of SA polymorphisms, and the demographic fate of the populations (i.e., positive growth, or extinction).

%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Analyses} \label{sec:analyses}

Diverse eco-evolutionary outcomes are possible in the model, including fixation of either allele, balanced polymorphism, population growth or extinction, and even evolutionary rescue and suicide (e.g., see \citealt{CaswellEtAl2018,deVriesCaswell2019a,deVriesCaswell2019b}). We focus on identifying parameter conditions where two criteria are satisfied: ({\itshape i}) SA polymorphism is maintained under balancing selection and ({\itshape ii}) the intrinsic population growth rate at equilibrium is positive; a situation that we refer to as a 'demographically viable SA polymorphism'.

We identify conditions where SA polymorphism is 'protected' by evaluating the stability of populations initially fixed for either SA allele to invasion by the other (i.e., stability was assessed at the boundary equilibrium allele frequencies of $\hat{\mbf{p}}_{AA} = 1$ and $\hat{\mbf{p}}_{aa} = 1$; \citealt{Levene1953, Prout1968, deVriesCaswell2019b}). The formal conditions for a protected polymorphism are determined by linearizing the model in the vacinity of the boundary equilibria ($\hat{\mbf{p}}$), and evaluating the magnitude of the largest eigenvalue of the Jacobian matrix of the linearization, which we denote $\tilde{\zeta}_{AA}$ and $\tilde{\zeta}_{aa}$. For our model, $\mbf{M}$ has an upper block triangular form, and we can therefore express the stability of $\hat{\mbf{p}}$ in terms of the leading eigenvalue of the diagonal block corresponding to changes in heterozygote frequency ($\mbf{M}_{22}$; e.g., see \citealt{OttoDay2007,deVriesCaswell2019b}). For a polymorphism to be 'protected', both boundary equilibria must be unstable (i.e., $\tilde{\zeta}_{AA} > 1$ and $\tilde{\zeta}_{aa} > 1$), which requires that
\begin{linenomath*}
\begin{equation}\label{eq:PolyCond}
	1 < \frac{\rho(\mbf{M}_{22})}{\lambda_{i}}
\end{equation}
\end{linenomath*}

\noindent is satisfied, where $\rho(\cdot)$ represents the spectral radius, and $\lambda_i$ is the intrinsic growth rate for the homozygous genotype being invaded at each boundary ($i \in (AA,aa)$; see Box.~1). A full derivation of the Jacobian and details of the invasion analysis are provided in \hl{Online Appendix X.x}.

We used numerical simulation to determine whether a protected SA polymorphism was also demographically viable. Specifically, for each boundary equilibrium we introduced the rare allele at low initial frequency, and iterated Eq(\ref{eq:Atilde}) until the population had reached demographic and genotypic equilibrium. In our model, the population state vector will grow or shrink exponentially after converging to stable population structure and genotypic frequencies (see \citealt{Caswell2001}), and the intrinsic population growth rate, $\lambda$, can be calculated as $\tilde{\mbf{n}}(t)/\tilde{\mbf{n}}(t-1)$, where $t$ is the final generation of the simulation. We note, however, that if the ecological component of the model is non-linear, more exotic dynamics are possible.

Unfortunately, closed-form solutions for the invasion conditions for each allele and the equilibrium frequencies under different selection scenarios are analytically intractable except for very restricted conditions (see \hl{Online Appendix X}). We therefore used numerical techniques to identify invasion and extinction thresholds, which could then be used to define regions of demographically viable polymorphism across $s_f \times s_m$ parameter space. Because single-locus selection coefficients are generally weak, we limit our analyses to coefficients with within the open interval $\left(0,0.15\right)$, unless stated otherwise. For a given parameter set, we numerically evaluated Eq(\ref{eq:PolyCond}) and $\lambda$, starting at the outer boundaries of selection parameter space, and titrated inward to identify thresholds where $\tilde{\zeta}_i \approx 1$ and $\lambda \approx 1$ (with precision of $10^{-5}$). The proportions of total $s_f \times s_m$ parameter space where SA polymorphism and/or extinction occurred were calculated by numerically integrating the appropriate regions defined by the invasion and extinction thresholds


%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Results}
%%%%%%%%%%%%%%%%%%%%%%%%
\ldots


%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Polymorphism and Extinction} \label{sec:Res:PolyExt}

 \begin{figure}[htbp]
 \centering
 \includegraphics[width=\linewidth]{../output/figs/extinctionThresholdsFig.pdf}
 \caption{\footnotesize{Illustration of parameter space for SA polymorphism and extinction thresholds predicted by the mendelian stage-structured matrix model. Balanced SA polymorphisms can be maintained in the funnel-shaped region between the invasion conditions for each SA allele (dark solid lines). However, for some parameter conditions, populations will ultimately go extinct (red shaded regions) due to reduced female fitness resulting from the male-beneficial/female-deleterious allele that is either segregating as a balanced polymorphism (inside the funnel), or becomes fixed (area below the funnel). We are particularly interested in identifying and quantifying "demographically viable polymorphic parameter space", which corresponds to the area inside the funnel that is also to the left of the extinction threshold for a given fertility value. Results are shown for three different population selfing rates ($C = \{0,\,1/4,\,1/2\}$), and two dominance scenarios (additivity with $h = 1/2$, and dominance reversal with $h = 1/4$); extinction thresholds are illustrated for three different values of female fecundity ($f$ values annotated on each panel).}}
 \label{fig:extThresholds}
 \end{figure}




%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Polymorphism in the absence of inbreeding depression} \label{sec:Res:PolyNoID}

 \begin{figure}[htbp]
 \centering
 \includegraphics[scale=0.75]{../output/figs/polymorphicSpaceTitrate.pdf}
 \caption{\footnotesize{Proportion of demographically viable parameter space (out of total $s_f \times s_m$ space with $\max(s) = 0.15$) in the absence of inbreeding depression (i.e., assuming $\delta = \delta_i = 0$, where $i \in \{j,a,\gamma\}$), plotted as a function of the population selfing rate. Results are shown for three fertility values corresponding to low, medium, and high fertility (blue, green, and red points respectively) under additive (panel A; $h = 1/2$), and partially recessive (panel B; $h = 1/4$) SA fitness effects. Each point was calculated by numerical integration of the corresponding SA invasion conditions and extinction threshold predicted by the mendelian matrix model (see \hyperref[sec:analyses]{Analyses} section), while solid lines were produced by numerically integrating the analytic expressions for the single-locus invasion conditions from the population genetic models of \citet{JordanConnallon2014} and \citet{Olito2017} (solid black lines).}} 
 \label{fig:polySpace}
 \end{figure}



%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Demographic effects of inbreeding depression} \label{sec:Res:InbreedEffects}

 \begin{figure}[htbp]
 \centering
 \includegraphics[scale=0.75]{../output/figs/deltaPolymorphicSpaceTitrate.pdf}
 \caption{\footnotesize{Effects of early- and late-acting inbreeding depression on the proportion of demographically viable parameter space (out of total $s_f \times s_m$ space with $\max(s) = 0.15$), plotted as a function of the population selfing rate. In all plots, the strength of inbreeding depression decreases as the selfing rate goes up following a simple model of purging recessive deleterious mutations (see \hl{Appendix X.x} for details). Only single, $\delta_i$ terms indicated in the legend ($i \in \{j,a,\gamma\}$) are allowed to vary at once (all others are set to $0$). Effects of early-acting inbreeding depression are shown on the left (panels A, C; $\delta$) and for late-acting inbreeding depression on the right (panels B, D; $\delta_i$) for additive ($h = 1/2$) and partially recessive ($h = 1/4$) SA fitness effects, respectively. All results used the same 'high female fertility' value of $f = 6.5$ (see Fig.\ref{fig:polySpace}). Each point was calculated by numerically integrating the corresponding SA invasion conditions and extinction threshold predicted by the model (see \hyperref[sec:analyses]{Analyses} section), while solid lines were produced by numerically integrating the analytic expressions for the single-locus invasion conditions from the population genetic models of \citet{JordanConnallon2014} and \citet{Olito2017} (solid black lines). The population genetic model predictions for varying $delta$ are plotted on both left- and right-hand panels for comparison, and to highlight the different y-axes scales.}} 
 \label{fig:deltaPolySpace}
 \end{figure}



%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Discussion}
%%%%%%%%%%%%%%%%%%%%%%%%

\ldots

%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Conclusion}
%%%%%%%%%%%%%%%%%%%%%%%%

\ldots

%%%%%%%%%%%%%%%%%%%%%
% Acknowledgments
%%%%%%%%%%%%%%%%%%%%%

%\section*{Acknowledgments}

% ... We are particularly grateful to the European Society for Evolutionary Biology (ESEB) for funding the Special Topics Network workshops “Linking local adaptation with the evolution of sex differences”, and to the participants, all of whom made this study possible. We would also like to thank .... This work was supported by a Wenner-Gren Foundation postdoctoral stipend to C.O., and [Lotte's funding] to C.dV..

\newpage{}

\section*{Appendix A: Supplementary Figures}

% In many cases, The American Naturalist allows authors to typeset 
% their own supplementary material in an author-supplied PDF. For author-
% supplied PDFs, please consult the AmNat_supp_template.tex document,
% available from https://www.journals.uchicago.edu/journals/an/instruct 
%
% By contrast, the Appendix instructions below apply to cases in which
% supplementary material is to be typeset by the AmNat editorial staff.
% That notably includes descriptions of methods, tables defining parameters,
% and other material necessary for reproducing the MS's results.
%
% Please reset counters for the appendix (thus normally figure A1, 
% figure A2, table A1, etc.).
%
% In certain cases, it may be appropriate to have a PRINT appendix in
% addition to (or instead of) an online appendix. In this case, please 
% name the print appendix Appendix A, and any subsequent appendixes (if 
% there are any) should be named Online Appendix B, Online Appendix C,
% etc.
%
% Counters for each appendix should match the letter of that appendix.
% For example, tables in Appendix C should be numbered table C1, table C2,
% etc. This applies to tables, equations, and figures.
%
% It's better not to use the \appendix command, because we have some
% formatting peculiarities that \appendix conflicts with.

\renewcommand{\theequation}{A\arabic{equation}}
\renewcommand{\thetable}{A\arabic{table}}
\setcounter{equation}{0}  % reset counter 
\setcounter{figure}{0}
\setcounter{table}{0}

\subsection*{Fox--dog encounters through the ages}

The quick red fox jumps over the lazy brown dog. The quick red fox has always jumped over the lazy brown dog. The quick red fox began jumping over the lazy brown dog in the 19th century and has never ceased from so jumping, as we shall see in figure~\ref{Fig:Jumps}. But there can be surprises (figure~\ref{Fig:JumpsOk}).

If the order and location of figures is not otherwise clear, feel free to include explanatory dummy text like this:

[Figure A1 goes here.]

[Figure A2 goes here.]

\subsection*{Further insights}

Tables in the appendices can appear in the appendix text (see table~\ref{Table:Rivers} for an example), unlike appendix figure legends which should be grouped at the end of the document together with the other figure legends.

\begin{table}[h]
\caption{Various rivers, cities, and animals}
\label{Table:Rivers}
\centering
\begin{tabular}{lll}\hline
River        & City        & Animal            \\ \hline
Chicago      & Chicago     & Raccoon           \\
Des Plaines  & Joliet      & Coyote            \\
Illinois     & Peoria      & Cardinal          \\
Kankakee     & Bourbonnais & White-tailed deer \\
Mississippi  & Galena      & Bald eagle        \\ \hline
\end{tabular}
\bigskip{}
\\
{\footnotesize Note: See table~\ref{Table:Founders} below for further table formatting hints.}
\end{table}

Lorem ipsum dolor sit amet, as we have seen in figures~\ref{Fig:Jumps} and \ref{Fig:JumpsOk}.

\newpage{}
\renewcommand{\theequation}{B\arabic{equation}}
% redefine the command that creates the equation number.
\renewcommand{\thetable}{B\arabic{table}}
\setcounter{equation}{0}  % reset counter 
\setcounter{table}{0}

\section*{Appendix B: Additional Methods}

\subsection*{Measuring the height of fox jumps without a meterstick}

Pellentesque ac nibh placerat, luctus lectus non, elementum mauris. 
Morbi odio velit, eleifend ut hendrerit vitae, consequat sit amet 
nulla. Pellentesque porttitor vitae nisl quis tempus. Pellentesque 
habitant morbi tristique senectus et netus et malesuada fames ac 
turpis egestas. Praesent ut nisi odio. Vivamus vel lorem gravida 
odio molestie volutpat condimentum et arcu. 

\begin{equation}
{ \frac{1}{N_k-1} \sum \limits_{t=1}^{N_k} (M_{tjk} - \bar{M}_{jk})^2}
\end{equation}

\subsection*{Quantifying the brownness of the dog}

Pellentesque eu nulla odio. Nulla aliquam porta metus, quis malesuada orci faucibus quis. Suspendisse nunc magna, tristique sit amet sollicitudin nec, elementum et lacus. Sed vitae elementum mi. In hac habitasse platea dictumst. Etiam eu tortor elit. Sed ac tortor purus. Aliquam volutpat, odio sit amet posuere pretium, dolor ex interdum ante, sed luctus quam eros ac nulla. 

\begin{equation}
{ (\sum \limits_{p=1}^P {n_{sp}})^{-1}\sum \limits_{p=1}^P {n_{sp}Q_{p}}}
\end{equation}

\newpage{}

%%%%%%%%%%%%%%%%%%%%%
% Bibliography
%%%%%%%%%%%%%%%%%%%%%
% You can either type your references following the examples below, or
% compile your BiBTeX database and paste the contents of your .bbl file
% here. The amnatnat.bst style file should work for this---but please
% let us know if you run into any hitches with it!
%
% If you upload a .bib file with your submission, please upload the .bbl
% file as well; this will be required for typesetting.
%
% The list below includes sample journal articles, book chapters, and
% Dryad references.
\bibliography{Refs2.bib}

\newpage{}



%%%%%%%%%%%%%%%%%%%%%
% Tables
%%%%%%%%%%%%%%%%%%%%%
\section*{Tables}
\renewcommand{\thetable}{\arabic{table}}
\setcounter{table}{0}


%%%%%%%%%%%%%%%%%%%%%%%%
\begin{table}[htbp]
\centering
\caption{\bf Definition of terms.}
\label{tab:Terms}
\begin{tabular}{ l l r }
 \toprule
Symbol & Definition & Dimension \\
\hline
$g$      & number of genotypes ($3$; $AA$, $Aa$, and $aa$) & \\
$\omega$ & number of stages ($2$; juvenile and adult) & \\
$\tilde{\mbf{n}}$ & Joint stage $\times$ genotype vector & $2 \omega g \times 1$ \\
$\tilde{\mbf{p}}$ & Joint stage $\times$ genotype frequency vector & $2 \omega g \times 1$ \\

$\mbf{U}^{S}_{i}$, $\mbf{U}^{X}_{i}$ & Genotype-specific transition and survival matrices & $\omega \times \omega$ \\
$\mbf{F}_{i}$, $\mbf{F}^{\prime}_{i}$ & Genotype-specific fertility matrices & $\omega \times \omega$ \\
$\mbf{H}^{S}_{i}$, $\mbf{H}^{X}_{i}$ & Parent-offspring genotype maps & $g \times g$ \\
$\mbf{W}$   & Allele segregation matrix & $\omega \times g$ \\
$\mbf{K}$   & Vec-permutation matrix & $g \times g$ \\
$\mbf{I}$   & Identity matrix & given in subscript \\
\hline
\end{tabular}
\end{table}
\newpage{}




\begin{table}[h]
\caption{Founders of \textit{The~American Naturalist}}
\label{Table:Founders}
\centering
\begin{tabular}{lll}\hline
Early editor            & Years with the journal \\ \hline
Alpheus S. Packard Jr.  & 1867--1886 \\
Frederick W. Putnam     & 1867--1874 \\ 
Edward S. Morse         & 1867--1871 \\ 
Alpheus Hyatt           & 1867--1871 \\
Edward Drinker Cope$^a$ & 1878--1897 \\
J.~S. Kingsley          & 1887--1896 \\ \hline 
\end{tabular}
\bigskip{}
\\
{\footnotesize Note: Table titles should be short. Further details should go in a `notes' area after the tabular environment, like this. $^a$ Published the first description of \textit{Dimetrodon}.}
\end{table}

\newpage{}

\section*{Figure legends}

\begin{figure}[h!]
%\includegraphics{horn-of-okapi}
\caption{Figure legends can be longer than the titles of tables. However, they should not be excessively long.}
\label{Fig:OkapiHorn}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%
% Videos
%%%%%%%%%%%%%%%%%%%%%
% If you have videos, journal style for them is similar to that for
% figures. You'll want to include a still image (such as a JPEG)
% to give your readers a preview of what the video looks like.

%%%%% Include the text below if you have videos

\renewcommand{\figurename}{Video} 
\setcounter{figure}{0}
% Thanks to Flo Debarre for the pro tip of putting
% \renewcommand{\figurename}{Video} before the Video legend and
% \renewcommand{\figurename}{Figure} after it!

\begin{figure}[h!]
%\includegraphics{VideoScreengrab.jpg}
\caption{Video legends can follow the same principles as figure legends. Counters should be set and reset so that videos and figures are enumerated separately.}
\label{VideoExample}
\end{figure}

\renewcommand{\figurename}{Figure}
\setcounter{figure}{1}

%%%%% Include the above if you have videos


\begin{figure}[h!]
%\includegraphics{elegance}
\caption{In this way, figure legends can be listed at the end of the document, with references that work, even though the graphic itself should be included for final files after acceptance. Instead, upload the relevant figure files separately to Editorial Manager; Editorial Manager should insert them at the end of the PDF automatically.}
\label{Fig:AnotherFigure}
\end{figure}

\subsection*{Online figure legends}

\renewcommand{\thefigure}{A\arabic{figure}}
\setcounter{figure}{0}

\begin{figure}[h!]
%\includegraphics{jumps20m}
\caption{\textit{A}, the quick red fox proceeding to jump 20~m straight into the air over not one, but several lazy dogs. \textit{B}, the quick red fox landing gracefully despite the skepticism of naysayers.}
\label{Fig:Jumps}
\end{figure}

\begin{figure}[h!]
%\includegraphics{jumps20m}
\caption{The quicker the red fox jumps, the likelier it is to land near an okapi. For further details,.}
\label{Fig:JumpsOk}
\end{figure}

\renewcommand{\thefigure}{B\arabic{figure}}
\setcounter{figure}{0}

\end{document}
